<!doctype html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="description" content="Fruit Fly Brain Observatory">
  <meta name="author" content="FFBO Team">

  <title>Get Involved | Workshops</title>

  <!-- Bootstrap Core CSS -->
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <link href="css/ffbo.css" rel="stylesheet">
  <link rel="shortcut icon" href="ico/favicon.ico" />
</head>

<body>
  <!-- PRELOADING  -->
  <!--div id="preload">
    <div class="preload">
      <div class="loader">
      </div>
      <h2>loading ...</h2>
    </div>
  </div-->

  <!-- Navigation -->
  <nav class="navbar navbar-fixed-top navbar-ffbo top-nav-collapse" role="navigation">
      <div class="container">
          <div data-scroll-header class="navbar-header">
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav">
                  <span class="sr-only">Toggle navigation</span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
              </button>
              <a class="navbar-brand" href="index.html"><span><img width="50px" height="50px" src="img/ffbo_logo.png"></span> Fruit Fly Brain Observatory</a>
          </div>
          <div class="collapse navbar-collapse" id="nav">
              <ul class="nav navbar-nav navbar-right uppercase">
                  <li>
                    <a class="dropdown-toggle" role="button" data-toggle="dropdown" href="#">Get Started</a>
                    <ul class="dropdown-menu" role="menu">
                      <li class="dropdown-submenu">
                          <a href="#" class="dropdown-toggle" data-toggle="dropdown">About</a>
                          <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#mission-2">Background</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#overview-2">Overview</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neuronlp-2">NeuroNLP</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neurogfx-2">NeuroGFX</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neuroapp-2">NeuroAPPs</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#under-the-hood-2">Under The Hood</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#team-2">About Us</a></li>
                          </ul>
                      </li>
                      <li class="divider"></li>
                      <li class="dropdown-submenu">
                        <a tabindex="-1" href="#"  class="dropdown-toggle" data-toggle="dropdown">Get Involved</a>
                        <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" href="faq.html" target="_blank">FAQs</a></li>
                          <li><a tabindex="-1" href="code.html" target="_blank">Code</a></li>
                          <li><a tabindex="-1" href="hackathons.html" target="_blank">Hackathons</a></li>
                          <li><a tabindex="-1" href="https://lists.columbia.edu/mailman/listinfo/ffbo" target="_blank">Mailing List</a></li>
                          <li><a tabindex="-1" href="workshops.html" target="_blank">Workshops</a></li>
                          <li><a tabindex="-1" href="https://github.com/NeuralEnsemble/NeuroinformaticsTutorial/blob/master/Exercises/Exercise6_FruitFlyBrainObservatory.md" target="_blank">Tutorials</a></li>
                        </ul>
                      </li>
                    </ul>
                  </li>
                  <li><a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroNLP</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="index.html#neuronlp-2">What is NeuroNLP</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neuronlp.fruitflybrain.org" target="_blank">Launch NeuroNLP.Adult</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neuronlp.larva.fruitflybrain.org" target="_blank">Launch NeuroNLP.Larva</a></li>
                      </ul>
                  </li>
                  <li><a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroGFX</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="index.html#neurogfx-2">What is NeuroGFX</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neurogfx.fruitflybrain.org" target="_blank">Launch NeuroGFX</a></li>
                      </ul>
                  </li>

                  <li>
                      <a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroAPPs</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="index.html#neuroapp-2">What are NeuroAPPs</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/retinal_degeneration"  target="_blank">Retinal Degeneration</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/parkinsons/olfaction" target="_blank">Parkinson's Disease: Olfaction</a></li>
                          <li><a  href="https://neuroapps.fruitflybrain.org/parkinsons/vision" target="_blank">Parkinson's Disease: Vision</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/epilepsy" target="_blank">Epilepsy</a></li>
                      </ul>
                  </li>
              </ul>
          </div>
          <!-- /.navbar-collapse -->
      </div>
      <!-- /.container -->
  </nav>


    <div class="container" style="padding-top:100px;padding-left:40px">
      <div class="col-lg-12">
        <h2>Columbia Workshop on Brain Circuits, Memory and Computation 2019</h2>
        <h4>BCMC 2019
        <br>March 21-22, 2019
        <br>Davis Auditorium, CEPSR
        <br>Columbia University, New York, NY, USA</h4><br>

        <h4>Overview</h4>
        <p>The goal of the workshop is to bring together researchers interested in developing executable models of
neural computation/processing of the brain of model organisms.
Of interest are models of computation that consist of elementary units
of processing using brain circuits and memory elements. Elementary units of computation/processing
include population encoding/decoding circuits with biophysically-grounded neuron models, non-linear dendritic processors for motion detection/direction selectivity, spike processing and pattern recognition neural circuits,
movement control and decision-making circuits, etc.
Memory units include models of spatio-temporal memory circuits, circuit models for memory access and
storage, etc.
A major aim of the workshop is to explore the integration of various sensory and control circuits in higher brain centers.
        </p>


        <p>A <a href="http://fruitflybrain.org/hackathons.html">Fruit Fly Brain Hackathon</a> is being conducted in conjunction with the workshop. Workshop participants are welcome to attend the hackathon.</p>
        <hr />

        <h4>Organizer and Program Chair</h4>
        <p><a href="http://www.ee.columbia.edu/~aurel">Aurel A. Lazar</a>, Columbia University</p>

        <hr />

        <h4>Registration</h4>
        <p>Registration is free but all participants have to <a href="http://bcmc2019.eventbrite.com">register</a>. Thank you!</p>

        <hr />

        <h4>Lodging and Directions to Venue</h4>
        <p>Please follow this <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2019/directions">link</a> for lodging details and directions to the hotel and venue.</p>

        <hr />

        <h4>Sponsorship</h4>

        <p>The 2019 Columbia Workshop on Brain Circuits, Memory and Computation is supported by the</p>
        <p><a class="reference external" href="http://www.ee.columbia.edu">Department of Electrical Engineering</a>, Columbia University</p>
        <p><a class="reference external" href="https://datascience.columbia.edu/computing-systems">Center for Computing Systems for Data-Driven Science</a>, Data Science Institute, Columbia University</p>
        <p><a class="reference external" href="http://www.engineering.columbia.edu">School of Engineering and Applied Science</a>, Columbia University</p>
        <hr />

        <h4>Program Overview</h4>
        (<a href="http://bionet.github.io/papers/bcmc_program19.pdf">PDF version including the list of close by restaurants</a>)
        <hr class="docutils" />
        <h5>Thursday, 9:00 AM - 5:30 PM, March 21, 2019</h5>
        <hr class="docutils" />
        <h5>Morning Session: The Inner Workings of the Fruit Fly Brain (I) (9:00 AM - 10:30 AM)</h5>
        <h5>Chair: Gwyneth Card</h5>
        <hr class="docutils" />
        <p>9:00 - 9:45 AM</p>
    <p><strong>The Somnostat: Mechanisms for Balancing Sleep Need and Sleep</strong></p>
    <a href="http://www.cncb.ox.ac.uk/the-science/research-groups/miesenboeck-group/" target="_blank">Gero A. Miesenboeck</a>, Centre for Neural Circuits and Behaviour, University of Oxford.
    <br />
    <p></p>
    <p>Sleep is vital and universal, but its biological function remains unknown. We seek to understand why we need to sleep by studying how the brain responds to sleep loss. Our studies in Drosophila have pinpointed neurons whose sleep-inducing activity switches on as sleep deficits accrue, revealed how this activity switch works, and furnished a molecular interpretation of sleep pressure, its accumulation, and its discharge.</p>
        <hr class="docutils" />
        <p>9:45 - 10:30 AM</p>
    <p><strong>Circuits that Encode and Predict Alcohol Associated Preference</strong></p>
    <a href="https://www.kaunlab.com" target="_blank">Karla Kaun</a>, Department of Neuroscience, Brown University.
    <br />
    <p></p>
    <p>The ability to associate a rewarding stimulus with a sensory cue from the environment is critical for an animal’s ability to find food and mates. Mapping the circuits in which these associations are formed, then initiate an output response provides a scaffold for understanding how experiences can influence decisions. Drugs of abuse such as alcohol can induce lasting changes in these circuits to induce maladaptive behavioral decisions. Here we describe a how memories for the intoxicating properties of alcohol are acquired and expressed through different mushroom body circuits. Acquisition of odor-alcohol memories induce lasting molecular changes which affect plasticity within circuits important for memory expression. Expression of these memories requires a remarkably complex multi-level circuit whereby dopamine directly, and indirectly via the mushroom body, modulates the activity of glutamatergic and cholinergic output neurons. Moreover, trans-synaptic tracing the outputs of these neurons suggests convergent and divergent networks, providing an elaborate framework for integrating external content and internal state to coordinate an appropriate output response. Together this work provides a snapshot of how alcohol can affect the dynamic molecular and circuit mechanisms required for behavioral decisions.</p>
    <p>Joint work with Kristin M. Scaplen<sup>1</sup>, Mustafa Talay<sup>2</sup>, Emily Petruccelli<sup>3</sup>, Nicolas Ledru<sup>4</sup>, Gilad Barnea<sup>1</sup>,
    <sup>1</sup>Department of Neuroscience, Brown University, Providence, RI.
    <sup>2</sup>Department of Molecular and Cell Biology, Harvard University, Cambridge, MA.
    <sup>3</sup>Department of Biology, Southern Illinois University, Edwardsville, IL.
    <sup>4</sup>MSTP Program, Washington University, St Louis, MO.</p>
    <p>Funding support: NIAAA R01AA024434, NIGMS P20GM103645, and the Smith Family Award for Excellence in Biomedical Research.</p>

    <hr class="docutils" />

    <h5>Morning Break 10:30 AM - 11:00 AM</h5>

    <hr class="docutils" />
        <h5>Morning Session: The Inner Workings of the Fruit Fly Brain (II) (11:00 AM - 12:30 PM)</h5>
        <h5>Chair: Kevin Franks</h5>

     <hr class="docutils" />

             <p>11:00 - 11:45 AM</p>
    <p><strong>Thermosensing in the Fly: from Genes to Cells to Behavior</strong></p>
    <a href="http://www.bio.brandeis.edu/garritylab/Welcome.html" target="_blank">Paul A. Garrity</a>, Department of Biology, Brandeis University.
    <br />
    <p></p>
    <p>Temperature is a universal physical variable that affects all aspects of physiology. Among other vital functions, animals rely on their thermosensory systems to maintain appropriate body temperatures and avoid thermal extremes. Thermosensing in Drosophila melanogaster depends on multiple classes of thermosensors, which rely on diverse classes of molecular thermoreceptors, exhibit distinct thermal sensitivities and have distinct behavioral functions.  How these thermosensory neurons encode temperature information and how the information they provide supports behavior remain largely open questions.  I will present recent work in which we have  identified a critical role for phasic thermosensors (sensory neurons which primarily respond to heating and cooling rather than hot or cold temperatures) in innocuous thermosensation in Drosophila. At the molecular level, we have identified the molecular receptors that confer thermosensitivity upon these neurons and found that these molecules are critically important for the morphogenesis of the elaborate sensory compartment involved in sensing temperature changes. At the behavioral level, we find that these phasic thermosensors are essential for mediating behavioral responses across a wide range of temperatures, consistent with their ability to respond robustly to small temperature changes at very different ambient temperatures. Together these findings begin to provide a consistent picture of thermosensory processing in the fly brain and raise questions concerning how phasic inputs might combine with other sensory inputs to drive behavior.</p>
           <hr class="docutils" />
           <p>11:45 - 12:30 PM</p>
    <p><strong>Olfactory Evolution in Drosophilids: Receptors, Neurons and Behaviours</strong></p>
    <a href="http://www.unil.ch/cig/benton" target="_blank">Richard Benton</a>, Center for Integrative Genomics, University of Lausanne.
    <br />
    <p></p>
    <p>My group is interested in understanding the structure, function and evolution of neural circuits. We exploit the olfactory system of Drosophila melanogaster as a model, which is well-described, experimentally accessible and dynamically evolving. Furthermore, genomic and growing genetic access to closely related, but ecologically diverse, drosophilids provides an unparalleled foundation for comparative studies of their olfactory circuits. I will present our recent work on the evolution of the olfactory pathways of Drosophila sechellia, an island endemic that displays extreme specialisation for the Morinda citrifolia “noni” fruit as a food source and breeding site. We have started to define the molecular basis by which D. sechellia’s olfactory receptors are re-tuned to the odors of its host and – through development of novel neurogenetic tools in this species – how these different sensory pathways contribute to host-seeking behaviors.</p>
    <hr class="docutils" />

    <h5>Lunch Break 12:30 PM - 2:00 PM</h5>

    <hr class="docutils" />
        <h5>Afternoon Session:  Neuroscience and Big Data (2:00 PM - 3:30 PM)</h5>
        <h5>Chair: Yiyin Zhou</h5>

        <hr class="docutils" />
        <p>2:00 - 2:45 PM</p>
    <p><strong>Better Connectome Reconstruction from Large Electron and Light
Microscopy Volumes of the Drosophila Brain</strong></p>
    <a href="https://www.janelia.org/lab/saalfeld-lab" target="_blank">Stephan Saalfeld</a>, HHMI Janelia Research Campus, Ashburn, VA.
    <br />
    <p></p>
    <p>At the resolution necessary to unambiguously extract the synaptic
connectome, the brain of the fruit fly Drosophila melanogaster has a
size of about 100 trillion voxels.  Today, 3D electron microscopy (3D-
EM) is the only available imaging method that provides this resolution.</p>
<p>Automatic reconstruction of neurons and synaptic connections has
seen significant improvement over the last few years, in particular
thanks to notable advances in machine learning.  Our contribution to
this improvement is threefold:</p>
<p>(1) We have generated the largest existing training data set for neuron
reconstruction and synaptic partner detection from non-isotropic 3D-EM
of the Drosophila brain.  With secret test data of equal size, we have
organized an ongoing public challenge for connectivity reconstruction
in the Drosophila brain (<a href="http://cremi.org" target="_blank">http://cremi.org</a>).
(2) Together with collaborators, we have developed the leading neuron
segmentation method for this dataset.  Further evaluations and
improvements are currently underway.
(3) We developed the leading method for synaptic cleft detection and
reconstructed all synaptic clefts in 3D-EM of the complete Drosophila
brain.  We generated a new, high-resolution statistical atlas of the
Drosophila brain from resolution limited confocal microscopy images
visualizing all synapses in the neuropils of the fly, and we
automatically registered the synapse cloud of the EM volume with the
light microscopy atlas such that neurons reconstructed in EM can be
matched with sparsely labeled light microscopy images.</p>
<p>I will give a brief overview of the state of automatic connectome
reconstruction in the fly brain and will discuss our ongoing efforts to
improve upon the current state.</p>
    <hr class="docutils" />
    <p>2:45 - 3:30 PM</p>
    <p><strong>Data-Driven Modeling of the Cortex Based on a Systematic Experimental Platform</strong></p>
    <a href="https://www.alleninstitute.org/what-we-do/brain-science/about/team/staff-profiles/anton-arkhipov/" target="_blank">Anton Arkhipov</a>, Allen Institute of Brain Science.
        <br />
    <p></p>
    <p>The Mindscope project at the Allen Institute for Brain Science aims to elucidate mechanisms underlying cortical function in the mouse, focusing on the visual system.  This involves concerted efforts of multiple teams characterizing cell types, connectivity, and neuronal activity in behaving animals.  An integral part of these efforts is the construction of models of the cortical tissue and cortical computations.  We will discuss our current progress on this front.  First, we built a computational pipeline to produce models of individual neurons based on slice electrophysiology and morphology reconstructions.  Second, we used these models as building blocks for large simulations of cortical activity in response to visual stimuli.  A highly realistic 230,000-neuron model of the mouse cortical area V1, receiving thalamocortical visual inputs, has been constructed at two levels of resolution: one using the biophysically detailed, compartmental neuron models and the other using point-neuron models.  Third, we perform systematic comparisons of simulated responses to in vivo experiments and investigate the structure-function relationships in the models to make mechanistic predictions for experimental testing.  To enable this work, we developed the software suite called Brain Modeling ToolKit (BMTK) and a modeling file format called SONATA.  These tools, the models, and simulation results are all being made freely available to the community via the Allen Institute Modeling Portal.</p>
    <hr class="docutils" />

    <h5>Afternoon Break 3:30 PM - 4:00 PM</h5>

    <hr class="docutils" />
        <h5>Afternoon Session: Neural Circuits and Behavior (4:00 PM - 5:30 PM)</h5>
        <h5>Chair: Tim Jarsky</h5>


     <hr class="docutils" />
        </p>4:00 - 4:45 PM<p>
    <p><strong>Using Machine Vision and Learning to Discover How the Brain Generates Behavior</strong></p>
    <a href="https://www.janelia.org/lab/branson-lab" target="_blank">Kristin Branson</a>, HHMI Janelia Research Campus, Ashburn, VA.
    <br />
    <p></p>
    <p>In this talk, I will overview our lab's work using machine vision and learning to analyze animal behavior from video in large-scale neuroscience experiments, including pose tracking, supervised behavior classification, and unsupervised behavior analysis.</p>

        <hr class="docutils" />
        </p>4:45 - 5:30 PM<p>
    <p><strong>The Neural Circuit Basis of Behavioral Individuality</strong></p>
    <a href="http://lab.debivort.org" target="_blank">Benjamin L. de Bivort</a>, Department of Organismic and Evolutionary Biology, Harvard University.
    <br />
    <p></p>
    <p>Individuals animals vary in their behaviors even when their genetics and environment are held constant. The mechanisms underlying this variation is still largely uncharacterized, though we have made some progress in understanding genetic and circuit variants that lead a population of animals to exhibit high or low variability in behavior. These are key insights, but fall short of predicting the specific behavioral biases of individual animals. We term the causal biological features that determine individual behavioral biases &quot;loci of individuality,&quot; and we have begun to search for them in the circuits that mediate sensory-evoked and spontaneous behaviors. We have found neural circuit elements, whose morphological properties predict behavioral biases. Specifically, the volume of axonal output arbors of central complex neurons that project to the Lateral Accessory Lobe correlates with changes in locomotor behavior in specific sensory contexts. We hypothesize that individual wiring variation in these neurons has a large effect on behavior because they lie at a bottleneck in the sensorimotor circuit, where stochastic fluctuations have an outsized effect on circuit outputs. Thus, we have found that individual variation in the structure of small numbers of neurons, in topologically critical circuit positions, predict individual behavioral biases in a sensory-context specific fashion.</p>

    <hr class="docutils" />
        <h5>Friday, 9:00 AM - 5:30 PM, March 22, 2019</h5>
        <hr class="docutils" />
        <h5>Morning Session: From Neural Circuits to Information Processing to Behavior (9:00 AM - 10:30 AM)</h5>
        <h5>Chair: Karla Kaun</h5>

     <hr class="docutils" />
     </p>9:00 - 9:45 AM<p>
    <p><strong>Neural Circuits Underlying Sex-Specific Behaviours</strong></p>
    <a href="http://www.cncb.ox.ac.uk/people/stephen-goodwin/" target="_blank">Stephen F. Goodwin</a>, Centre for Neural Circuits and Behaviour, University of Oxford.
    <br />
    <p></p>
    <p>All animals must continuously sequence and coordinate behaviours appropriate to both their environment and internal state if they are to survive and reproduce. Dissecting the neural substrates that initiate, organize, and terminate these behavioural sequences is critical to understanding behaviour. Courtship behaviour in Drosophila has proven to be an outstanding model for understanding how a compact circuit coordinates these critical action sequences. Our group is interested in understanding how neural circuits are functionally assembled, and how they integrate information to generate appropriate sex-specific responses.</p>
    <p>In the fly, sex-specific behaviours are hardwired into the nervous system via the actions of two sex determination transcription factors (TFs), Doublesex (Dsx) and Fruitless (Fru). We have focused our attention on neurons that express these TFs to find anatomical or molecular sex differences in neuronal populations in order to gain an entry point into the neural circuits underlying gender-typical behaviours and identify the neuronal nodes that control component behaviours and behavioural sequencing. We have focused on the role of neurons that express dsx. These dsx+ neurons control male courtship behaviour and aspects of female receptivity. Yet little is known about the specific role of dsx+ neurons in the brain, which are believed to control mating decisions. We identified a group of sexually-dimorphic dsx neurons that are critical to sensory integration in males and females. Importantly, the dimorphism we uncovered can alter connectivity and information flow between males and females.</p>
    <hr class="docutils" />
    </p>9:45 - 10:30 AM<p>
    <p><strong>Towards a Brain Architecture for Visual Behavior Selection</strong></p>
    <a href="https://www.janelia.org/lab/card-lab" target="_blank">Gwyneth Card</a>, HHMI Janelia Research Campus, Ashburn, VA.
    <br />
    <p></p>
    <p>Selecting the right behavior at the right time is critical for animal survival.  Animals rely on their senses to deliver information about the environment to sensory processing areas in the brain that extract relevant features and form the perceptual representations that guide behavior.  We aim to uncover the organization of this feature space and the neural mechanisms by which these cues are translated into dynamic motor activity.</p>
    <p>Our current focus is visually-driven behaviors of the fly.  In particular, those driven by visual looming cues produced by an approaching predator or an imminent collision.  The same looming stimulus can evoke a wide range of different behaviors, including a rapid escape jump, a slower, more stable takeoff sequence, or a landing response.  We use whole-cell patch clamp physiology in behaving flies, calcium imaging, high-throughput/high-resolution behavioral assays, and genetic tools to examine the transformation of information from sensory to motor. I will discuss our recent work investigating the representation of ethologically-relevant visual features in the fly optic glomeruli and the mechanisms by which descending neurons read out this feature information to produce an appropriate behavioral choice.</p>

     <hr class="docutils" />

    <h5>Morning Break 10:30 AM - 11:00 AM</h5>

    <hr class="docutils" />
        <h5>Morning Session: Olfactory Coding and Odor Signal Processing  (11:00 AM - 12:30 PM)</h5>
        <h5>Chair: Kristin Branson</h5>

        <hr class="docutils" />
        </p>11:00 - 11:45 AM<p>
<p><strong>Decoding and Demixing Smells</strong></p>
    <a href="https://vnmurthylab.org" target="_blank">Venkatesh N. Murthy</a>, Department of Molecular and Cellular Biology, Harvard University.
    <br />
    <p></p>
    <p>Fluctuating mixtures of odorants, often transported in fluid environments, are detected by an array of chemical sensors and parsed by neural circuits to recognize odor objects that can inform behavioral decisions. Whether and how mice segment odor mixtures into individual components (“demix”) remains unclear. We have found that mice can be trained to recognize individual odorants embedded in unpredictable and variable background mixtures with high degree of success [<a href="https://doi.org/10.1038/nn.3775" target="_blank">1</a>]. Despite nonlinear interactions and variability in the representations of odor mixtures by odorant receptors [<a href="https://doi.org/10.7554/eLife.34958" target="_blank">2</a>], a simple linear feedforward decoding is sufficient to explain the performance of mice in this task [<a href="https://doi.org/10.1016/j.neuron.2016.08.007" target="_blank">3</a>]. Current experiments are aimed at understanding how the mouse brain represents information about odor mixtures to aid odor object identification and categorization.</p>

     <hr class="docutils" />
     </p>11:45 - 12:30 PM<p>
     <p><strong>Neural Circuits for Odor Coding in Piriform Cortex</strong></p>
    <a href="http://www.thefrankslab.org" target="_blank">Kevin M. Franks</a>, Department of Neurobiology, Duke University.
    <br />
    <p></p>
    <p>Animals rely on olfaction to find food, attract mates and avoid predators. Odor information is initially encoded as combinations of co-active olfactory bulb glomeruli, which each represent a distinct molecular feature of the odorant. This elemental odor code must then be integrated and processed to form a synthetic or gestalt odor percept.  Much of this processing is thought to occur in the primary olfactory (piriform) cortex.  In this talk I will present some of our recent work describing how piriform cortex transforms olfactory bulb input. We show that piriform performs computations such as decorrelation, intensity invariance and pattern completion that support the formation of stable and distinct cortical odor representations, and we dissect the neural circuit processes that implement these operations.</p>
    <hr class="docutils" />

    <h5>Lunch Break 12:30 PM - 2:00 PM</h5>

    <hr class="docutils" />
        <h5>Afternoon Session: Neural Circuits: From Structure to Function (I) (2:00 PM - 3:30 PM)</h5>
        <h5>Chair: Anton Arkhipov</h5>

         <hr class="docutils" />
     </p>2:00 - 2:45 PM<p>
    <p><strong>Building the Functional Map of the Fruit Fly Brain</strong></p>
    <a href="https://www.bionet.ee.columbia.edu" target="_blank">Aurel A. Lazar</a>, Columbia University, New York, NY.
    <br />
    <p></p>
    <p>I will describe the open interactive computing platform, called <a href="http://www.bionet.ee.columbia.edu/research/ffbo/fbl" target="_blank">FlyBrainLab</a> (FBL), designed for studying the function of executable circuits constructed from fly brain data. FBL provides easy access to biological data and
    brain circuit models of the <a href="http://fruitflybrain.org" target="_blank">Fruit Fly Brain Observatory</a> (FFBO).
    FFBO lists a (i) hub for storing and integrating fruit fly brain data from multiple data sources worldwide,
    (ii) unified repository of tools and methods to build, emulate and compare brain models in health and disease, and (iii) an open framework for brain data processing and model execution.
    Furthermore, FFBO provides access to application tools for visualizing, configuring, simulating and analyzing computational models of brain circuits of the cell-type map, connectome, synaptome, and activity map using intuitive queries in plain English. FBL code and installation instructions are available on <a href="http://github.com/flybrainlab/flybrainlab" targe="_blank">Github</a>.</p>


     <hr class="docutils" />
     </p>2:45 - 3:30 PM<p>
<p><strong>Connecting the Structure and Function of Neural Circuits</strong></p>
    <a href="https://www.janelia.org/lab/turaga-lab" target="_blank">Srinivas C. Turaga</a>, HHMI Janelia Research Campus, Ashburn, VA.
    <br />
    <p></p>
    <p>In this talk, I will describe how we developed deep learning based computational tools to solve two problems in neuroscience: inferring the activity of a neural network from measurements of its structural connectivity, and inferring the connectivity of a network of neurons from measurements and perturbation of neural activity.</p>
    <ol class="arabic simple">
    <li>Can we infer neural connectivity from noisy measurement and perturbation of neural activity? Population neural activity measurement by calcium imaging can be combined with cellular resolution optogenetic activity perturbations to enable the mapping of neural connectivity in vivo. This requires accurate inference of perturbed and unperturbed neural activity from calcium imaging measurements, which are noisy and indirect. We built on recent advances in variational autoencoders to develop a new fully Bayesian approach to jointly inferring spiking activity and neural connectivity from in vivo all-optical perturbation experiments. Our model produces excellent spike inferences and predicts connectivity for mouse primary visual cortex which is consistent with known measurements.</li>
    <li>Are measurements of the structural connectivity of a biological neural network sufficient to predict its function? We constructed a simplified model of the first two stages of the fruit fly visual system, the lamina and medulla. The result is a deep hexagonal lattice convolutional neural network which discovered well-known orientation and direction selectivity properties in T4 neurons and their inputs. Our work demonstrates how knowledge of precise neural connectivity, combined with knowledge of the function of the circuit, can enable in silico predictions of the functional properties of individual neurons in a circuit, leading to an understanding of circuit function from structure.</li>
    </ol>

   <hr class="docutils" />

    <h5>Afternoon Break 3:30 PM - 4:00 PM</h5>

    <hr class="docutils" />
        <h5>Afternoon Session: Neural Circuits: From Structure to Function (II) (4:00 PM - 5:30 PM)</h5>
        <h5>Chair: Benjamin L. de Bivort</h5>

        <hr class="docutils" />
        </p>4:00 - 4:45 PM<p>
        <p><strong>Microcircuitry of the Cortex: Connectivity, Strength, and Short-Term Plasticity</strong></p>
    <p><a href="https://www.alleninstitute.org/what-we-do/brain-science/about/team/staff-profiles/tim-jarsky/" target="_blank">Tim Jarsky</a>, Allen Institute of Brain Science.
    <br />
    <p></p>
    <p>We seek to characterize the microcircuitry of the cortex in mouse and human using multipatch electrophysiology supplemented with 2-photon optogenetic circuit mapping. Using the approaches mentioned above, we quantify connection probability as a function of distance, synaptic strength, and short-term synaptic dynamics among the major classes of cortical neurons. We rely on the dimensionality reduction that modeling can provide to quantify and compare differences in synaptic dynamics. We intend to share the data so that accurate, integrative computer models of cortex can be made. This presentation will summarize our results to date and share how our data may be accessed.</p>



     <hr class="docutils" />
     </p>4:45 - 5:30 PM<p>
    <p><strong>Completing the Fly Model?</strong></p>
    <a href="https://www.janelia.org/lab/scheffer-lab" target="_blank">Louis Scheffer</a>, HHMI Janelia Research Campus, Ashburn, VA.
    <br />
    <p></p>
    <p>We are on the verge of having a full connectome of all neurons and their chemical synapses in Drosophila.  But while necessary, this is not sufficient to model the fly's nervous system.  Additional information includes gap junction varieties and locations, identities of neurotransmitters, receptor types and locations, neuromodulators and hormones (with sources and receptors), the role of glial cells, time evolution rules for synapses, and more.   This talk will outline these missing pieces, look at methods by which the data might be obtained, and summarize existing efforts (where they exist) to get this needed information.</p>

        <hr />

        <h4>More information about BCMC 2019 can be found <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2019">here.</a></h4>
        <br>

        <h4>Past BCMC workshops</h4>

        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2018">BCMC 2018</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2017">BCMC 2017</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2016">BCMC 2016</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2015">BCMC 2015</a></p>
        <br>

      </div>
    </div>


  <footer class="footer">
  <div class="row">
    <p>Copyright &copy; 2018 <a href="licenses.html#credits">FFBO Teams</a>. All Rights Reserved. <a href="licenses.html">Licenses and Credits</a></p>
  </div>
  </footer>

  <!-- jQuery Version 1.12.3 -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
  <!-- Bootstrap Core JavaScript -->
  <script src="js/bootstrap.min.js"></script>
  <!-- Custom Modernizr JavaScript -->
  <script src="js/modernizr.custom.js"></script>
  <script src="js/smooth-scroll.js"></script>
  <script src="js/jquery.stellar.min.js"></script>
  <script src="js/submenu.js"></script>
<!--  <script src="js/ffbo.js"></script>-->
</body>
</html>
