<!doctype html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="description" content="Fruit Fly Brain Observatory">
  <meta name="author" content="FFBO Team">

  <title>Get Involved | Workshops</title>

  <!-- Bootstrap Core CSS -->
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <link href="css/ffbo.css" rel="stylesheet">
  <link rel="shortcut icon" href="ico/favicon.ico" />
</head>

<body>
  <!-- PRELOADING  -->
  <!--div id="preload">
    <div class="preload">
      <div class="loader">
      </div>
      <h2>loading ...</h2>
    </div>
  </div-->

  <!-- Navigation -->
  <nav class="navbar navbar-fixed-top navbar-ffbo top-nav-collapse" role="navigation">
      <div class="container">
          <div data-scroll-header class="navbar-header">
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav">
                  <span class="sr-only">Toggle navigation</span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
              </button>
              <a class="navbar-brand" href="index.html"><span><img width="50px" height="50px" src="img/ffbo_logo.png"></span> Fruit Fly Brain Observatory</a>
          </div>
          <div class="collapse navbar-collapse" id="nav">
              <ul class="nav navbar-nav navbar-right uppercase">
                  <li>
                    <a class="dropdown-toggle" role="button" data-toggle="dropdown" href="#">Get Started</a>
                    <ul class="dropdown-menu" role="menu">
                      <li class="dropdown-submenu">
                          <a href="#" class="dropdown-toggle" data-toggle="dropdown">About</a>
                          <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#mission-2">Background</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#overview-2">Overview</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neuronlp-2">NeuroNLP</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neurogfx-2">NeuroGFX</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neuroapp-2">NeuroAPPs</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#under-the-hood-2">Under The Hood</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#team-2">Fruit Fly Brain Explorers</a></li>
                          </ul>
                      </li>
                      <li class="divider"></li>
                      <li class="dropdown-submenu">
                        <a tabindex="-1" href="#"  class="dropdown-toggle" data-toggle="dropdown">Get Involved</a>
                        <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" href="faq.html" target="_blank">FAQs</a></li>
                          <li><a tabindex="-1" href="hackathons.html" target="_blank">Hackathons</a></li>
                          <li><a tabindex="-1" href="https://lists.columbia.edu/mailman/listinfo/ffbo" target="_blank">Mailing List</a></li>
                          <li><a tabindex="-1" href="workshops.html" target="_blank">Workshops</a></li>
                          <li><a tabindex="-1" href="https://github.com/NeuralEnsemble/NeuroinformaticsTutorial/blob/master/Exercises/Exercise6_FruitFlyBrainObservatory.md" target="_blank">Tutorials</a></li>
                        </ul>
                      </li>
                    </ul>
                  </li>
                  <li><a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroNLP</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a href="https://neuronlp.fruitflybrain.org" target="_blank">NeuroNLP.Adult</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neuronlp.larva.fruitflybrain.org" target="_blank">NeuroNLP.Larva</a></li>
                      </ul>
                  </li>
                  <li><a href="https://neurogfx.fruitflybrain.org" target="_blank">NeuroGFX</a></li>
                  <li>
                      <a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroAPPs</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a href="https://neuroapps.fruitflybrain.org/retinal_degeneration"  target="_blank">Retinal Degeneration</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/parkinsons/olfaction" target="_blank">Parkinson's Disease: Olfaction</a></li>
                          <li><a  href="https://neuroapps.fruitflybrain.org/parkinsons/vision" target="_blank">Parkinson's Disease: Vision</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/epilepsy" target="_blank">Epilepsy</a></li>
                      </ul>
                  </li>
              </ul>
          </div>
          <!-- /.navbar-collapse -->
      </div>
      <!-- /.container -->
  </nav>


    <div class="container" style="padding-top:100px;padding-left:40px">
      <div class="col-lg-12">
        <h2>Columbia Workshop on Brain Circuits, Memory and Computation 2018</h2>
        <h4>BCMC 2018
        <br>March 15-16, 2018
        <br>Center for Neural Engineering and Computation
        <br>Columbia University, New York, NY, USA</h4><br>

        <h4>Overview</h4>
        <p>The goal of the workshop is to bring together researchers interested in developing executable models of
neural computation/processing of the brain of model organisms.
Of interest are models of computation that consist of elementary units
of processing using brain circuits and memory elements. Elementary units of computation/processing
include population encoding/decoding circuits with biophysically-grounded neuron models, non-linear dendritic processors for motion detection/direction selectivity, spike processing and pattern recognition neural circuits,
movement control and decision-making circuits, etc.
Memory units include models of spatio-temporal memory circuits, circuit models for memory access and
storage, etc.
A major aim of the workshop is to explore the integration of various sensory and control circuits in higher brain centers.
        </p>


        <p>A <a href="http://www.bionet.ee.columbia.edu/hackathons/ffbh/2018">Fruit Fly Brain Hackathon</a> is being conducted in conjunction with the workshop. Workshop participants are welcome to attend the hackathon.</p>
        <hr />

        <h4>Organizer and Program Chair</h4>
        <p><a href="http://www.ee.columbia.edu/~aurel">Aurel A. Lazar</a>, Columbia University</p>

        <hr />

        <h4>Registration</h4>
        <p>Registration is free but all participants have to <a href="http://bcmc2018.eventbrite.com">register</a>. Thank you!</p>

        <hr />

        <h4>Lodging and Directions to Venue</h4>
        <p>Please follow this <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2018/directions">link</a> for lodging details and directions to the hotel and venue.</p>

        <hr />

        <h4>Program Overview</h4>
        (<a href="http://bionet.github.io/papers/bcmc_program18.pdf">PDF version including the list of close by restaurants</a>)
        <hr class="docutils" />
        <h5>Thursday, 9:00 AM - 5:30 PM, March 15, 2018</h5>
        <hr class="docutils" />
        <h5>Morning Session: Olfactory Coding and Odor Signal Processing (9:00 AM - 10:30 AM)</h5>
        <h5>Chair: Pavan R. Ramdya</h5>
        <hr class="docutils" />
        </p>9:00 - 9:45 AM<p>
    <p><strong>Universality in Olfactory Coding of Odor Identity and Intensity</strong></p>
    <a href="https://scholar.harvard.edu/aravisamuel/home" target="_blank">Aravinthan D. T. Samuel</a>, Harvard University, Cambridge, MA
    <br />
    </p>Odor perception serves multiple purposes: to distinguish odorants, recognize the same odorant across concentrations, and determine concentration changes in space and time. How primary olfactory representations are structured to support and disentangle these distinct functions remains poorly understood. By interrogating the complete olfactory input circuit of the Drosophila larva, we have uncovered a set of simple cellular an ensemble-level strategies that allow for non-overlapping representations of odor identity and intensity. We find that the activity of each olfactory neuron scales with the concentration of any odorant via a fixed dose-response function, but varies in its activation threshold across ORNs. Furthermore, all activation thresholds are drawn from the same power law statistical distribution. These properties predict a universal scaling relationship between odor intensity and the overall ensemble response across olfactory space. We find that similar temporal response filters across odorants and ORNs extend these invariant relationships to fluctuating or turbulent environments. The activation threshold is thus the only free variable quantifying the tuning between any odorant and receptor. We conclude that the diverse tuning properties and dynamics of individual olfactory neurons function to produce a prescribed statistical outcome in the olfactory code. We propose that a set of invariances exhibited by the olfactory input circuit - from single ORN to ensemble representations - constitute efficient strategies for invariant and distinct representations of odor identity and intensity.
        <hr class="docutils" />
        </p>9:45 - 10:30 AM<p>
    <p><strong>Mixtures Are More Robust Stimuli in Olfaction</strong></p>
    <a href="http://users.sussex.ac.uk/~tn41/" target="_blank">Thomas Nowotny</a>, University of Sussex, Brighton, UK
    <br />
    <p>In natural environments, odors are typically mixtures of several different chemical compounds. However, the implications of mixtures for odor processing have not been fully investigated. We extended a standard olfactory receptor model to mixtures and found through its mathematical analysis that the first-spike latencies of receptor neurons are shorter and activity patterns are more stable across concentrations for mixtures. Shorter first-spike latencies arise from the nonlinear dependence of binding rate on odorant concentration, commonly described by the Hill coefficient, while the more stable activity patterns result from competition between different ligands for free receptor sites. We then directly demonstrated, in numerical simulations and in physiological recordings in insects, that both mixture effects occur in olfactory receptor neurons and central neurons. Our results suggest that olfactory systems respond to mixtures more robustly than to single odorants, which resonates with the observation that chemical signaling in animals often involves mixtures.
    <hr class="docutils" />
    
    <h5>Morning Break 10:30 AM - 11:00 AM</h5>
    
    <hr class="docutils" />
        <h5>Morning Session: Olfactory Learning and Memory I (11:00 AM - 12:30 PM)</h5>
        <h5>Chair: Hokto Kazama</h5>

     <hr class="docutils" />
     
             </p>11:00 - 11:45 AM<p>
    <p><strong>Competition between Memories of Opposite Valence Underlies Memory Extinction in Drosophila</strong></p>
    <a href="https://www.dpag.ox.ac.uk/research/waddell-group" target="_blank">Scott Waddell</a>, University of Oxford, Oxford, UK
    <br />
    </p>Animals constantly reassess the reliability of learned information to optimize their behavior. On retrieval, memory can be neutralized by extinction if the learned prediction was inaccurate. In Drosophila learning establishes a valence specific imbalance in the mushroom body output neuron network, which when reactivated during memory retrieval allows flies to re-evaluate what they have previously learned. Extinction of sugar-reinforced memory requires output neurons with dendrites in the vertical lobes of the mushroom body, which drive negatively reinforcing dopaminergic neurons that innervate neighbouring zones. The aversive valence of these new extinction memories neutralizes previously learned odour preference. In contrast, extinction of aversive electric shock-reinforced memory requires output neurons with dendrites in the horizontal lobes of the mushroom body, which drive positively reinforcing dopaminergic neurons that innervate the same zones. The new parallel competing extinction 'reward' memory can be seen to co-exist with the original aversive memory and to be integrated in the mushroom body output network to direct behaviour according to the fly?s most up-to-date knowledge.
    </p>Joint work with Johannes Felsenberg, Pedro F. Jacob, Tom Walker, Oliver Barnstedt, Amelia Edmondson-Stait, Markus W. Pleijzer, Nils Otto, Philipp Schlegel, Nadiya Sharifi, Emmanuel Perisse, Carlas S. Smith, J. Scott Lauritzen, Marta Costa, Gregory S. X. E. Jefferis   and Davi Bock.
           <hr class="docutils" />
           </p>11:45 - 12:30 PM<p>
    <p><strong>Olfactory Learning Alters Neuronal Gain in the Memory-Encoding Mushroom Body</strong></p>
    <a href="http://www.scripps.edu/tomchik/" target="_blank">Seth Tomchik</a>, Scripps Research Institute, Jupiter, Florida
    <br />
    </p>Learning and memory generate changes in neuronal responses to input stimuli, which are collectively referred to as a memory trace or engram. These alterations rely on dopaminergic circuits and downstream cAMP-dependent plasticity across diverse organisms. Yet it is not well understood how cAMP-dependent plasticity drives coherent changes in neuronal physiology that encode memory. In Drosophila, the mushroom body (MB) is critically involved in olfactory learning, and cAMP signaling molecules are necessary for normal memory in intrinsic MB neurons. The MB encodes a sparse representation of olfactory space, which is computationally advantageous for learning. Our recent studies have suggested that olfactory classical conditioning alters the gain of MB neuronal responses to sensory stimuli in a cAMP-dependent manner. This was observed particularly during appetitive conditioning, suggesting that it may either include a motivational salience component or reflect a bias toward reward learning. These data link the physiology of MB neurons to the behavioral roles for cAMP signaling molecules in learning and memory. I will discuss these results and their implications for development of models of olfactory learning and MB function.
    <hr class="docutils" />
    
    <h5>Lunch Break 12:30 PM - 2:00 PM</h5>
    
    <hr class="docutils" />
        <h5>Afternoon Session: Development of  Retinotopic Maps and Feature Detection (2:00 PM - 3:30 PM)</h5>
        <h5>Chair: Donggen Luo</h5>

        <hr class="docutils" />
        </p>2:00 - 2:45 PM<p>
    <p><strong>Development of Concurrent Retinotopic Maps in the Fly Motion Detection Circuit</strong></p>
    <a href="https://wp.nyu.edu/desplanlab/" target="_blank">Claude Desplan</a>, Center for Genomics and Systems Biology, NYU Abu Dhabi and Department of Biology, NYU.
    <br />
    </p>Understanding how complex brain wiring is produced during development is a daunting challenge. In the Drosophila optic lobe, information from each of 800 ommatidia is processed in distinct neuropiles, each subdivided into 800 matching retinotopic columns. In the motion vision pathway, T4 and T5 neurons respond to bright edge (T4) or dark edge (T5) motion. They collect information from the elementary motion detectors either in the Medulla (T4) or in the Lobula (T5). There are four T4 and four T5 subtypes, each tuned to one of four cardinal directions, back-to-front, front-to-back, up, or down, effectively establishing eight concurrent retinotopic maps in the Lobula Plate that detects wide-field motion. We will describe a mode of neurogenesis in which a neuroblast produces matching sets of four T4 and T5 neurons that are retinotopically coincident and have pair-wise opposite direction selectivity. Neuroblasts originating from a neuroepithelium expressing the signaling molecule Dpp produce T4 and T5 that detect vertical motion, while other neuroblasts originating from a neighboring region expressing the negative regulator of Dpp, Brinker, detect horizontal motion. We will show that retinotopy is an emergent characteristic of this neurogenic program that derives directly from neuronal birth order. This illustrates how simple developmental rules can implement complex neural organization.
        </p>Joint work with Filipe Pinto-Teixeira.
    <hr class="docutils" />
    </p>2:45 - 3:30 PM<p>
    <p><strong>Saccadic Feature Detection in Flies</strong></p>
    <a href="http://www.fryelab.net/" target="_blank">Mark A. Frye</a>, Department of Integrative Biology and Physiology, University of California, Los Angeles
        <br />
    </p>We studied visually-guided saccades and smooth optomotor movements in rigidly- and magnetically-tethered flying Drosophila. We show that flies use smooth pursuit optomotor movements to stabilize movement of the visual panorama, whereas body saccades enable orientation maneuvers to fixate a feature of interest or to avoid a threat. The amplitude, angular velocity, and torque transients of bar-fixation saccades were finely tuned to the speed of bar motion, and were triggered by a threshold in the temporal integral of the bar error angle, rather than its absolute retinal position error. A hybrid control model based on spatiotemporal integration simulates saccade trigger events and dynamics. We used two-photon excitation imaging to screen the major motion vision pathways within the fly brain for activity that matches the stimulus dynamics that trigger saccades. Three classes of lobula visual projection neuron encode spatial and temporal dynamics of the visual features that drive saccades, whereas horizontal motion detectors of the lobula plate do not show calcium accumulation in response to the same visual feature dynamics. 
    <hr class="docutils" />
    
    <h5>Afternoon Break 3:30 PM - 4:00 PM</h5>
    
    <hr class="docutils" />
        <h5>Afternoon Session: Auditory Processing (4:00 PM - 5:30 PM)</h5>
        <h5>Chair: Simon G. Sprecher</h5>

        <hr class="docutils" />
        </p>4:00 - 4:45 PM<p>
    <p><strong>Excitatory-Inhibitory Circuits in Auditory Processing</strong></p>
    <a href="https://geffenlab.weebly.com/" target="_blank">Maria N. Geffen</a>, University of Pennsylvania, Philadelphia, PA
    <br />
    </p>Hearing perception relies on our ability to tell apart the spectral content of different sounds, and to learn to use this difference to distinguish behaviorally relevant (such as dangerous and safe) sounds. However, the neuronal circuits that underlie this modulation remain unknown. In the auditory cortex, the excitatory neurons serve the dominant function in transmitting information about the sensory world within and across brain areas, whereas inhibitory interneurons carry a range of modulatory functions, shaping the way information is represented and processed. I will discuss the results of three of our recent studies that elucidate the function of specific inhibitory neuronal populations in sound encoding and perception. First, we found that inhibitory interneurons in the auditory cortex play a regulatory role in controlling a basic auditory behavior of frequency discrimination. Our results demonstrate that cortical inhibition can improve or impair acuity of innate and learned auditory behaviors. Second, we found that a specific type of cortical inhibitory neurons regulates adaptation in the auditory cortex to frequent sounds, in a stimulus-specific fashion. More recent experiments demonstrate that the role of these interneurons extends to other forms of adaptation to acoustic temporal regularities. Third, we identified that a center for emotional learning, the basolateral amygdala, gates cortical auditory responses via inhibition in the thalamic reticular. These results expand our understanding of how inhibitory-excitatory neuronal circuits contribute to auditory perception in everyday acoustic environments.
    
     <hr class="docutils" />
        </p>4:45 - 5:30 PM<p>
    <p><strong>Panel Discussion</strong></p>
    Moderator: Rudy Behnia, Columbia University
    
    <hr class="docutils" />
        <h5>Friday, 9:00 AM - 5:30 PM, March 16, 2018</h5>
        <hr class="docutils" />
        <h5>Morning Session: Connectomics (9:00 AM - 10:30 AM)</h5>
        <h5>Chair: Davi Bock</h5>
 
     <hr class="docutils" />
     </p>9:00 - 9:45 AM<p>
    <p><strong>Sequencing the Connectome</strong></p>
    <a href="http://zadorlab.cshl.edu/" target="_blank">Anthony M. Zador</a>, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
    <br />
</p>To study the connectivity of neural circuits at high resolution, we have developed an approach based on DNA sequencing that allows us to determine the connectivity of single neurons. The key idea is to tag each neuron with a random nucleotide sequence (a "barcode"), which can be read out by high-throughput sequencing. By recasting neuroanatomy, which is traditionally viewed as a problem of microscopy, as a problem of sequencing, MAPseq harnesses advances in sequencing to permit high-throughput interrogation of brain circuits. The ability to read out brain circuitry in a high-throughput way has the potential to accelerate our understanding of both normal brain function, and the causes underlying neuropsychiatric disorders including autism and schizophrenia.

    <hr class="docutils" />
    </p>9:45 - 10:30 AM<p>
    <p><strong>Connectome Coding</strong></p>
    <a href="http://jovo.me/" target="_blank">Joshua T. Vogelstein</a>, Johns Hopkins University, Baltimore, MD
    <br />
     <hr class="docutils" />
    
    <h5>Morning Break 10:30 AM - 11:00 AM</h5>
    
    <hr class="docutils" />
        <h5>Morning Session: Sensory Coding and CPGs (11:00 AM - 12:30 PM)</h5>
        <h5>Chair: Thomas Nowotny</h5>
    
        <hr class="docutils" />
        </p>11:00 - 11:45 AM<p>
<p><strong>Lessons from Simple Networks: Sensory Coding in the Drosophila Larval Visual System</strong></p>
    <a href="http://www.sprecherlab.com/simon-sprecher/" target="_blank">Simon Sprecher</a>, Department of Biology, University of Fribourg, Fribourg, Switzerland
    <br />
    </p>Neural circuits in the brain have the striking ability to encode sensory information with high precision, but also to adapt and modulate synaptic activity to allow memories to be formed. Cellular complexity in circuit organization is often a major obstacle when attempting to understand these processes. A comparably low number of neurons and powerful genetics allows us in the fruit fly Drosophila melanogaster to tackle fundamental topics in neuroscience with unprecedented precision. Combining connectomics, behavioural studies and in vivo life imaging we decipher visual and gustatory coding principles. Genetic manipulations with single-cell resolution allowed us to identify molecular mechanisms of multimodal and multisensory coding, supporting the existence of previously unknown gustatory coding principles. Similarly, by identifying the synaptic circuit map of the visual system and underlying function of individual neurons for visually guided behaviours we depict how spatial and temporal information is encoded, highlighting general principles of visual information processing.

     <hr class="docutils" />
     </p>11:45 - 12:30 PM<p>
     <p><strong>Neural Circuits that Mediate Visual Photoentrainment in Drosophila</strong></p>
    <a href="http://cqb.pku.edu.cn/en/kxdw/zxjs/DonggenLuo/254007.shtml" target="_blank">Donggen Luo</a>, Peking University, Beijing, China
    <br />
    </p>The central master circadian clock emerges from a network of circadian pacemaker neurons to orchestrate daily rhythms of physiology and behavior. An important feature of the master clock is its ability to synchronize its endogenous rhythm to external light/dark cycles through photoentrainment, but the precise circuit mechanisms underlying visual photoentrainment remain largely unknown. Here, we studied the neural circuits that mediate visual inputs to central circadian pacemaker neurons in Drosophila. We will show our mapping of the functional circuit connections between visual systems and central circadian pacemaker neurons.
    <hr class="docutils" />
    
    <h5>Lunch Break 12:30 PM - 2:00 PM</h5>

    <hr class="docutils" />
        <h5>Afternoon Session: Olfactory Learning and Memory II (2:00 PM - 3:30 PM)</h5>
        <h5>Chair: Seth Tomchik</h5>
        
         <hr class="docutils" />
     </p>2:00 - 2:45 PM<p>
    <p><strong>Randomness, Order, and Mystery in the Fly Mushroom Body</strong></p>
    <a href="https://www.janelia.org/lab/bock-lab" target="_blank">Davi Bock</a>, Janelia Research Campus, Ashburn, VA
    <br />
    </p>The mushroom body (MB) of the Drosophila olfactory system is critical for olfactory learning and memory. The MB on each side of the brain contains ~2,000 Kenyon cells (KCs) that receive olfactory inputs from projection neurons (PNs) in the MB calyx. Light microscopy (LM) data of PN-to-KC pairs pooled across many animals, as well as theoretical arguments, have suggested that the PN-to-KC synaptic network is completely random. However, the PN-to-KC network has never been mapped within a single animal. We used a whole-brain EM dataset to map the connections of PNs to ~20% of KCs in the MB on one side of the brain. We find that PN arbors are more tightly clustered in calyx than predicted from pooled LM data, and that KCs with neighboring output neurites receive more common PN input than predicted by chance. Thus the fly MB connectome, which has been considered to be random, has a clear structure. Further work will be needed to understand how this network structure impacts associative learning and recall in the fly brain.


     <hr class="docutils" />
     </p>2:45 - 3:30 PM<p>
<p><strong>Olfactory Computation in the Drosophila Mushroom Body: Beyond Sparsening and Decorrelation</strong></p>
    <a href="http://kazama.brain.riken.jp/" target="_blank">Hokto Kazama</a>,&nbsp;Riken Brain Science Institute, Saitama, Japan
    <br />
    </p>To understand sensory computations performed by a particular brain region, it is important to record neural activity from both the input and the target regions and that in a comprehensive manner. The latter aspect is especially critical for olfaction because odor information is distributed among a large number of neurons without clear chemotopy and thus examination of a partial representation may overlook the full functional capacity of a brain region. Here, we fulfilled this aim in the olfactory system of adult Drosophila by conducting comprehensive volumetric Ca2+ imaging and analyzing the responses of 37 out of ~50 glomeruli in the antennal lobe and all ~2,000 Kenyon cells in the mushroom body to a diverse set of odors. We will discuss characteristics of olfactory computation in the mushroom body beyond previously reported sparsening and decorrelation of odor representations. We will also show how an experimentally constrained mechanistic model can recapitulate the responses of individual Kenyon cells to odors.

   <hr class="docutils" />
    
    <h5>Afternoon Break 3:30 PM - 4:00 PM</h5>
    
    <hr class="docutils" />
        <h5>Afternoon Session: Modeling Motor Control (4:00 PM - 5:30 PM)</h5>
        <h5>Chair: Joshua T. Vogelstein</h5>

        <hr class="docutils" />
        </p>4:00 - 4:45 PM<p>
        <p><strong>Motor Preparation Through Rebound in an Identified Sensory Integrator</strong></p>
    <p><a href="https://www.janelia.org/lab/ahrens-lab" target="_blank">Misha B. Ahrens</a>, Janelia Research Campus, Ashburn, VA
    <br />
    </p>While some actions are triggered directly by instantaneous incoming sensory information, most take into account internal information streams that represent information accumulated over longer time periods. Despite decades of study, much is still unknown at the level of neural circuits about how the brain integrates sensory information in between actions, stores this information, and transmits it to inform future actions. We made use of modern whole-brain microscopy methods and the advantages of genetically modified zebrafish to search the entire brain, cell-by-cell, during behavior, for neural integrators involved in motor preparation. Larval zebrafish respond to visual motion, but swim in discrete swim bouts. Accordingly, visual motion must be integrated in between swim bouts to influence future motor output. Our whole-brain activity screen identified a single brain region that responds to visual flow, integrates and stores it in ongoing activity, and modulates future behavior. These neurons respond to stimuli that encourage the fish to swim, and integrate them so that temporally separated stimuli have an additive effect. Moreover, their activity levels are positively correlated with preceding levels of motor output. However, surprisingly, stimulating the neurons suppressed instantaneous swimming, yet advanced the onset time of future swims. Network models based on these results suggests a brainstem integrator network that, in between actions, stores visual information while actively suppressing motor output, and during actions transmits the stored information to premotor circuits through post-inhibitory rebound. Voltage imaging using novel voltage indicators revealed precise temporal dynamics of this population of neurons in relation to behavior consistent with this model. These findings suggest inhibitory integrators as a complementary alternative to integrate-to-bound models for sensory integration and motor decisions.
    
    

     <hr class="docutils" />
     </p>4:45 - 5:30 PM<p>
    <p><strong>Reverse-Engineering Drosophila Limb Control Circuits</strong></p>
    <a href="https://ramdya-lab.epfl.ch/" target="_blank">Pavan P. Ramdya</a>, EPFL, Lausanne, Switzerland
    <br />
    </p>A shared goal of neuroscience and robotics is to understand how systems can be built to move effectively through the world. However, state-of-the-art algorithms for selecting and executing limbed behaviors in robots are still quite primitive compared with those used by animals. To inform robotic control approaches, we are investigating how the fly, Drosophila melanogaster, walks, grooms, and reaches. I will discuss how we are combining 2-photon imaging of the ventral nerve cord in behaving Drosophila with physics-based simulations and neural network modeling to uncover how flies achieve flexible limb control.

        <hr />


        <h4>More information about BCMC 2018 can be found <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2018">here.</a></h4>
        <br>

        <hr />

        <h4>Past BCMC workshops</h4>

        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2017">BCMC 2017</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2016">BCMC 2016</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2015">BCMC 2015</a></p>
        <br>

      </div>
    </div>


  <footer class="footer">
  <div class="row">
    <p>Copyright &copy; 2018 <a href="licenses.html#credits">FFBO Teams</a>. All Rights Reserved. <a href="licenses.html">Licenses and Credits</a></p>
  </div>
  </footer>

  <!-- jQuery Version 1.12.3 -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
  <!-- Bootstrap Core JavaScript -->
  <script src="js/bootstrap.min.js"></script>
  <!-- Custom Modernizr JavaScript -->
  <script src="js/modernizr.custom.js"></script>
  <script src="js/smooth-scroll.js"></script>
  <script src="js/jquery.stellar.min.js"></script>
  <script src="js/submenu.js"></script>
<!--  <script src="js/ffbo.js"></script>-->
</body>
</html>
