<!doctype html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="description" content="Fruit Fly Brain Observatory">
  <meta name="author" content="FFBO Team">

  <title>Get Involved | Workshops</title>

  <!-- Bootstrap Core CSS -->
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <link href="css/ffbo.css" rel="stylesheet">
  <link rel="shortcut icon" href="ico/favicon.ico" />
</head>

<body>
  <!-- PRELOADING  -->
  <!--div id="preload">
    <div class="preload">
      <div class="loader">
      </div>
      <h2>loading ...</h2>
    </div>
  </div-->

  <!-- Navigation -->
  <nav class="navbar navbar-fixed-top navbar-ffbo top-nav-collapse" role="navigation">
      <div class="container">
          <div data-scroll-header class="navbar-header">
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav">
                  <span class="sr-only">Toggle navigation</span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
              </button>
              <a class="navbar-brand" href="index.html"><span><img width="50px" height="50px" src="img/ffbo_logo.png"></span> Fruit Fly Brain Observatory</a>
          </div>
          <div class="collapse navbar-collapse" id="nav">
              <ul class="nav navbar-nav navbar-right uppercase">
                  <li>
                    <a class="dropdown-toggle" role="button" data-toggle="dropdown" href="#">Get Started</a>
                    <ul class="dropdown-menu" role="menu">
                      <li class="dropdown-submenu">
                          <a href="#" class="dropdown-toggle" data-toggle="dropdown">About</a>
                          <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="#mission-2">Background</a></li>
                          <li><a tabindex="-1" data-scroll href="#overview-2">Overview</a></li>
                          <li><a tabindex="-1" data-scroll href="#neuronlp-2">NeuroNLP</a></li>
                          <li><a tabindex="-1" data-scroll href="#neurogfx-2">NeuroGFX</a></li>
                          <li><a tabindex="-1" data-scroll href="#neuroapp-2">NeuroAPPs</a></li>
                          <li><a tabindex="-1" data-scroll href="#under-the-hood-2">Under The Hood</a></li>
                          <li><a tabindex="-1" data-scroll href="#team-2">Fruit Fly Brain Explorers</a></li>
                          </ul>
                      </li>
                      <li class="divider"></li>
                      <li class="dropdown-submenu">
                        <a tabindex="-1" href="#"  class="dropdown-toggle" data-toggle="dropdown">Get Involved</a>
                        <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" href="faq.html" target="_blank">FAQs</a></li>
                          <li><a tabindex="-1" href="hackathons.html" target="_blank">Hackathons</a></li>
                          <li><a tabindex="-1" href="https://lists.columbia.edu/mailman/listinfo/ffbo" target="_blank">Mailing List</a></li>
                          <li><a tabindex="-1" href="workshops.html" target="_blank">Workshops</a></li>
                          <li><a tabindex="-1" href="https://github.com/NeuralEnsemble/NeuroinformaticsTutorial/blob/master/Exercises/Exercise6_FruitFlyBrainObservatory.md" target="_blank">Tutorials</a></li>
                        </ul>
                      </li>
                    </ul>
                  </li>
                  <li><a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroNLP</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a href="https://neuronlp.fruitflybrain.org" target="_blank">NeuroNLP.Adult</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neuronlp.larva.fruitflybrain.org" target="_blank">NeuroNLP.Larva</a></li>
                      </ul>
                  </li>
                  <li><a href="https://neurogfx.fruitflybrain.org" target="_blank">NeuroGFX</a></li>
                  <li>
                      <a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroAPPs</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a href="https://neuroapps.fruitflybrain.org/retinal_degeneration"  target="_blank">Retinal Degeneration</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/parkinsons/olfaction" target="_blank">Parkinson's Disease: Olfaction</a></li>
                          <li><a  href="https://neuroapps.fruitflybrain.org/parkinsons/vision" target="_blank">Parkinson's Disease: Vision</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/epilepsy" target="_blank">Epilepsy</a></li>
                      </ul>
                  </li>
              </ul>
          </div>
          <!-- /.navbar-collapse -->
      </div>
      <!-- /.container -->
  </nav>


    <div class="container" style="padding-top:100px;padding-left:40px">
      <div class="col-lg-12">
        <h2>Columbia Workshop on Brain Circuits, Memory and Computation 2017</h2>
        <h4>BCMC 2017
        <br>March 13-14, 2017
        <br>Center for Neural Engineering and Computation
        <br>Columbia University, New York, NY, USA</h4><br>

        <h4>Overview</h4>
        <p>The goal of the workshop is to bring together researchers interested in developing executable models of
        neural computation/processing of the brain of model organisms.
        Of interest are models of computation that consist of elementary units
        of processing using brain circuits and memory elements. Elementary units of computation/processing
        include population encoding/decoding circuits with biophysically-grounded neuron models, non-linear dendritic processors for motion detection/direction selectivity, spike processing and pattern recognition neural circuits,
        movement control and decision-making circuits, etc.
        Memory units include models of spatio-temporal memory circuits, circuit models for memory access and
        storage, etc.
        A major aim of the workshop is to explore the integration of various sensory and control circuits in higher brain centers.</p>
        <p>A <a href="http://www.bionet.ee.columbia.edu/hackathons/ffbh/2017">Fruit Fly Brain Hackathon</a> is being conducted in conjunction with the workshop. Workshop participants are welcome to attend the hackathon.</p>
        <br>

        <h4>Organizer and Program Chair</h4>
        <p><a href="http://www.ee.columbia.edu/~aurel">Aurel A. Lazar</a>, Department of Electrical Engineering, Columbia University.</p>
        <br>

        <h4>Registration</h4>
        <p>Registration is free but all participants have to <a href="http://bcmc17.eventbrite.com">register</a>. Thank you!</p>
        <br>


        <h4>Lodging and Directions to Venue</h4>
        <p>Please follow this <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2017/directions">link</a> for lodging details and directions to the hotel and venue.</p>
        <br>

        <h4>Program Overview</h4>
        <p>(<a href="http://bionet.github.io/papers/bcmc_program17.pdf">PDF version including the list of close by restaurants</a>)</p>

        <p><strong>Monday, 9:00 AM - 5:30 PM, March 13, 2017</strong></p>
        <hr />
        <p><strong>Morning Session: Learning and Memory I (9:00 AM - 10:30 AM)</strong></p>
        <p></p>
        <p><strong>Chair: Katherine I. Nagel</strong></p>
        <p></p>


        <hr />

        <p>9:00 AM - 9:45 AM</p>
        <p><strong>Circuits for Learning and Memory in the Adult Drosophila Mushroom Body</strong></p>
        <p><a href="http://www.hhmi.org/scientists/gerald-m-rubin">Gerald M. Rubin</a>, Janelia Research Campus, Ashburn, VA.</p>
        <p>The mushroom body (MB) is the major site of associative learning in insects. In the MB of adult Drosophila, each of ~20 types of dopaminergic neurons (DANs) innervates distinct small compartmental regions of the mushroom body (MB). Different subsets of these DANs signal punishment and reward. We have recently shown (Aso & Rubin, eLife 2016) that these DANs appear to be able to “write” parallel memories in different MB compartments using distinct “learning rules”. I will discuss our ongoing efforts to explore the rules for writing, forgetting and updating memories in each compartment.</p>
        <p>I will also present recent work done with the FlyEM Project Team at Janelia to determine the connectome of the three compartments of the vertical (or lobe) of the MB. The unprecedented level of detail of this dataset should enable modeling studies not previously possible and suggests many experiments to explore the physiological and behavioral significance of the circuit motifs we observed. That many of these motifs were not anticipated by over thirty years of extensive anatomical, experimental and theoretical studies on the role of the insect MB argues strongly for the value of electron microscopic connectomics studies.
        </p>

        <p></p>
        <hr />

        <p>9:45 AM - 10:30 AM</p>
        <p><strong>Long-Term Memory Requires Sequential Protein Synthesis in Discrete Mushroom Body Output Neurons in Drosophila</strong></p>
        <p><a href="http://brc.life.nthu.edu.tw/FlyLab/html/leader.html">Ann-Shyn Chiang</a>, National Tsing Hua University, Hsinchu, Taiwan.</p>
        <p>Creating long-term memory (LTM) requires new proteins to stabilize learning-induced synaptic changes in the brains. In the fruit flies, Drosophila melanogaster, aversive olfactory learning forms several phases of memory to associate an odor with coincident punishment in sparse Kenyon cells within the mushroom body (MB). How brain circuits translate early phases of labile memory into long-lasting stable memory remains unclear. Here, we show that learning-induced new proteins for aversive olfactory LTM occur at discrete MB output neurons (MBONs). Acutely blocking protein synthesis with a temperature-sensitive ribosomal toxin, we found that LTM formation requires sequential new proteins in three subsets of MBONs at different time windows after learning. RNAi-mediated down-regulation of oo18 RNA-binding proteins (ORB) in any of these MBONs impaired LTM. Neurotransmission outputs from these MBONs are required during specific time windows after learning and essential during LTM retrieval. Together, these results suggest a LTM formation model that early labile memory encoded in neural activities of sparse Kenyon cells consolidates into stable LTM at discrete postsynaptic MBONs by sequential ORBregulated local protein synthesis at active synapses.</p>
        <p>Joint work with Jie-Kai Wu, Chu-Yi Tai, Kuang-Lin Feng, Shiu-Ling Chen and Chun-Chao Chen.</p>

        <p></p>
        <hr />

        <p><strong>Morning Break 10:30 AM - 11:00 AM</strong></p>

        <p></p>
        <hr />

        <p><strong>Morning Session: Learning and Memory II (11:00 AM - 12:30 PM)</strong></p>
        <p></p>
        <p><strong>Chair: Marion Silies</strong></p>
        <p></p>


        <hr />

        <p>11:00 AM - 11:45 AM</p>
        <p><strong>Brain Circuit Maps of Larval Drosophila</strong></p>
        <p><a href="https://www.janelia.org/people/albert-cardona">Albert Cardona</a>, Janelia Research Campus, Ashburn, VA.</p>
        <p>The projection neurons of the olfactory, gustatory, visual and temperature-sensing systems of the insect relay inputs to both the mushroom body and the lateral horn. While quite a bit is known about the nature of the inputs, little is known about the circuits of these two brain centers and how they relate to each other. To address this, we have mapped the wiring diagram of these two centers using electron microscopy-based neuron reconstructions. I will show how multiple sensory modalities are integrated in the lateral horn and how this information is relayed to the mushroom bodies and beyond.</p>

        <p></p>
        <hr />

        <p>11:45 AM - 12:30 PM</p>
        <p><strong>The Larval Standard Brain of Drosophila: The Mushroom Body Learning and Memory Network</strong></p>
        <p><a href="https://cms.uni-konstanz.de/neuro/member/?tx_mhkleineidam_pi1%5BpId%5D=5029">Andreas S. Thum</a>, Department of Biology, University of Konstanz.</p>
        <p>Brains organize behavior. This involves the integration of present sensory input, past experience, and options for future behavior. The insect mushroom body is a paradigmatic case of a central brain structure bringing about such triadic integration. We use larval Drosophila to systematically study these processes at single-cell resolution. Our focus is bipartite as it includes research on the anatomical and behavioral architecture of the larval mushroom body circuit. On the anatomical level we use serial electron microscopy (EM) to reconstruct a synapse-resolution map (connectome) of the complete MB wiring diagram in a Drosophila larva consisting per hemisphere of about 110 KCs and exactly 24 mushroom body output neurons, 7 dopaminergic input neurons, 2 paired and 2 unpaired octopaminergic input neurons, 5 additional mushroom body modulatory input neurons, and an additional GABAergic feedback neuron. In addition, we study these mushroom body function with respect to appetitive olfactory learning at single-cell resolution, focusing on the behavioral architecture of the mushroom body input and output neurons, as well as the mushroom body intrinsic APL neuron. Ultimately, this data is then integrated in a newly established standard atlas for the larval brain, a five-part approach that includes the generation of an image registration framework, the generation of a larval standard brain, the segmentation and denomination of identified brain structures, the registration of several thousand Gal4 stocks onto the standard brain, and the organization of the obtained information in a web-based open access database called brainbase. Taken together this work provides a rich picture on multiple levels of how an insect central brain structure is anatomically and functionally organized.</p>

        <p></p>
        <hr />

        <p><strong>Lunch 12:30 PM - 2:00 PM</strong></p>

        <p></p>
        <hr />

        <p><strong>Afternoon Session: Olfactory Coding and Odor Signal Processing (2:00 PM - 3:30 PM)</strong></p>
        <p></p>
        <p><strong>Chair: M. Eugenia Chiappe</strong></p>
        <p></p>


        <hr />

        <p>2:00 PM - 2:45 PM</p>
        <p><strong>Parallel Olfactory Coding Mechanisms in the Drosophila Brain</strong></p>
        <p><a href="https://www.ice.mpg.de/ext/index.php?id=hopa&amp;pers=sisa3622">Silke Sachse</a>, Max Planck Institute for Chemical Ecology, Jena.</p>
        <p>Animals use sensory systems to navigate the environment in a way that optimizes their survival and reproduction. The olfactory system plays here a major role in encoding chemical information and translating the outside world into a neuronal representation that enables an animal to take odorguided decisions. The vinegar fly represents a premier model system for studying olfactory processing mechanisms since it exhibits a stereotyped architecture which is similar to its mammalian counterpart, but is less complex and highly tractable as well as susceptible to genetic manipulations. By exploiting these genetic techniques and linking them to neurophysiological, molecular and behavioral methods, we are dissecting the neural circuits underlying olfactory coding and processing in the context of odor-guided behavior in Drosophila.</p>
        <p>In order to understand higher odor processing we are scrutinizing the lateral horn (LH) of the protocerebrum, a brain region that is assumed to be involved in innate behavior. Two populations of projection neurons – excitatory (ePNs) and inhibitory (iPNs) – convey the odor information from the antennal lobe to the LH. We analyzed how different odor features such as hedonic valence and intensity are functionally integrated in this brain area. We could previously demonstrate that the LH can be classified into three functional odor response domains that decode opposing hedonic valences and odor intensity that derive from iPNs and third-order neurons that further innervate the ventro-lateral protocerebum. To investigate whether ePNs accomplish a comparable categorization of odor features and whether iPNs and ePNs interact, we have elucidated the neuronal circuitry of the two PN populations at a morphological and functional level. The talk will summarize our recent insights into the processing strategies of the two parallel output pathways to the higher brain and their contribution to odor perception.</p>

        <p></p>
        <hr />

        <p>2:45 PM - 3:30 PM</p>
        <p><strong>Understanding the Function and Specificity of Feedforward and Feedback Signals in Olfaction</strong></p>
        <p><a href="http://albeanulab.labsites.cshl.edu/people/">Dinu Florin Albeanu</a>, Cold Spring Harbor Laboratory, Cold  Spring Harbor, NY.</p>
        <p>Areas at the sensory periphery send feedforward signals to the cortex and, in turn, receive massive top-down cortical feedback. To date, the function and degree of specificity of such feedforward and feedback loops remain poorly understood. In the mammalian olfactory system, the bulb (OB) output neurons, the mitral and tufted (M/T) cells differ in their intrinsic properties and local connectivity, and project in a disjointed fashion to several areas including the piriform cortex (PC) and anterior olfactory nucleus (AON). In turn, both AON and PC send massive glutamatergic projections to local inhibitory OB interneurons (mainly granule cells, GC). To understand the logic of feedforward and feedback signals, we used multiphoton calcium imaging (GCaMP) and monitored the odor responses of M/T and GC cells in awake, head-fixed mice. We investigated whether mitral and tufted cells code odor identity and concentration differentially, and found substantial differences in their concentration response curve shapes, odor correlation patterns and dimensionality of responses. The cross-validated linear decoding performance for concentration-invariant odor identity was significantly better for the tufted versus the mitral cells population, suggesting distinct roles for these output channels in processing of olfactory information. Furthermore, we find that top-down feedback from higher olfactory brain areas depends on the functional type of feedforward input, and differentially impacts each of the two classes of OB outputs. We are currently investigating whether distinct sets of GCs synapsing on mitral vs. tufted cells are differentially modulated by feedback originating in the APC and AON. We propose that two neuronal circuits supported by the mitral and respectively tufted cells, their target brain areas and specific feedback signals, serve distinct computations in olfaction.</p>

        <p></p>
        <hr />

        <p><strong>Afternoon Break 3:30 PM - 4:00 PM</strong></p>

        <p></p>
        <hr />

        <p><strong>Afternoon Session: Coding and Deep Learning in Neuroscience (4:00 PM - 5:30 PM)</strong></p>
        <p></p>
        <p><strong>Chair: Tim P. Vogels</strong></p>
        <p></p>


        <hr />

        <p>4:00 PM - 4:45 PM</p>
        <p><strong>Cracking the Olfactory Code</strong></p>
        <p><a href="http://www.sas.upenn.edu/~vbalasub/public-html/Home.html">Vijay Balasubramanian</a>, Department of Physics, University of Pennsylvania.</p>
        <p></p>

        <p></p>
        <hr />

        <p>4:45 PM - 5:30 PM</p>
        <p><strong>Deep Learning and the Unknown Unknowns of Neuroscience</strong></p>
        <p><a href="http://klab.smpp.northwestern.edu/wiki/index.php5/People">Konrad P. Kording</a>, Feinberg School of Medicine, Northwestern University.</p>
        <p>I will highlight fundamental problems in neuroscience using both computational and experimental arguments. I will use this as a backdrop to discuss how neuroscience can be informed by deep learning, highlighting various ways in which deep learning can inform neuroscience.</p>

        <p></p>
        <hr />


        <p><strong>Tuesday, 9:00 AM - 5:30 PM, March 14, 2017</strong></p>
        <hr />
        <p><strong>Morning Session: Cognition and Behavior (9:00 AM - 10:30 AM)</strong></p>
        <p></p>
        <p><strong>Chair: Silke Sachse</strong></p>
        <p></p>


        <hr />

        <p>9:00 AM - 9:45 AM</p>
        <p><strong>Mating Circuits in the Drosophila Brain</strong></p>
        <p><a href="https://www.janelia.org/people/barry-dickson">Barry J. Dickson</a>, Janelia Research Campus, Ashburn, VA.</p>
        <p>We are investigating the neural circuits that control the sex-specific mating behaviors of Drosophila males and females. These circuits are shaped during the development by the fruitless (fru) and doublesex (dsx) genes. In the adult, these two genes together are believed to label most if not all of the neurons with sexually dimorphic functions in mating. They comprise a set of ~1500 neurons in both sexes, representing approximately ~150 morphologically distinct cell types. Several of these cells are sex-specific; many others are present in both sexes but have sexually dimorphic features, in some cases known to include sex-specific connectivity.</p>
        <p>I will present work that examines the organizational architecture of these circuits and how they contribute to the distinct mating behaviors of male and female flies.
        </p>

        <p></p>
        <hr />

        <p>9:45 AM - 10:30 AM</p>
        <p><strong>How Improvements in Neuroanatomy Could Shed Light on Cognitive Architecture</strong></p>
        <p><a href="http://www.adammarblestone.org">Adam H. Marblestone</a>, Synthetic Neurobiology Group, MIT.</p>
        <p>Neuroanatomy would benefit both from improved methods, and from computationally motivated theories that make distinctive predictions about mesoscale neural structures. I will describe an optical approach we are developing for high-throughput molecularly annotated connectomics, which treats long-range and short-range connections on an equal footing (a collaborative project among the Church, Boyden and Zador labs), introduce a theory of symbolic processing in the thalamo-cortico-striatal system (work with Ken Hayworth), and suggest how the former might in the future shed light on the latter. More generally, I will suggest neuroanatomical questions that could illuminate connections between biological brains and the kinds of structured architectures that are now finding use in deep learning.</p>

        <p></p>
        <hr />

        <p><strong>Morning Break 10:30 AM - 11:00 AM</strong></p>

        <p></p>
        <hr />

        <p><strong>Morning Session: Visual Circuits and Computation (11:00 AM - 12:30 PM)</strong></p>
        <p></p>
        <p><strong>Chair: Albert Cardona</strong></p>
        <p></p>


        <hr />

        <p>11:00 AM - 11:45 AM</p>
        <p><strong>Synaptic Mechanisms for Visual Computation in the Retina</strong></p>
        <p><a href="http://eyes.yale.edu/people/jonathan_demb.profile">Jonathan B. Demb</a>, Yale School of Medicine.</p>
        <p>Information processing in a sensory system depends on the selective convergence of excitatory and inhibitory pathways in neural circuitry as well as activity-dependent changes in synaptic release. The first stage of visual processing occurs in the retina, where circuits are formed by specific excitatory and inhibitory interneuron types that selectively converge onto ~40 types of ganglion cell, the output neurons that form the optic nerve. Ganglion cells adapt in order to operate under variable lighting conditions. Adaptation to both the mean light level and the variance around the mean, or the contrast, occurs beyond the photoreceptors within retinal circuitry. I will describe recent experiments that investigate mechanisms for adaptation in the circuitry of the mouse retina using receptive field measurements, optogenetic studies of specific synapses and computational modeling. Mechanisms for contrast adaptation depend on both synaptic mechanisms, including vesicle depletion, and network mechanisms, including lateral inhibition. Contrast adaptation in ganglion cell excitatory inputs can be explained by a divisive suppression computation that also contributes to millisecond precision of spike firing.</p>

        <p></p>
        <hr />

        <p>11:45 AM - 12:30 PM</p>
        <p><strong>Motion Vision: From Behavior to Cellular and Circuit Function</strong></p>
        <p><a href="http://www.eni-net.org/members/dr-marion-silies">Marion Silies</a>, European Neuroscience Institute, Göttingen.</p>
        <p>Many animals use visual motion cues to navigate through the environment, find prey or escape predators. Long-standing theoretical models have made predictions about the computations that compare light signals across space and time to detect motion. Over the past years, core circuits that can implement such motion computations in Drosophila have been proposed based on connectomic and physiological studies. In the fly visual system, separate ON and OFF pathway exist that are specialized to detect the movement of light or dark edges. These pathways already split downstream of photoreceptors in the lamina, where L1 and L2 provide the major inputs to the ON and OFF pathway, respectively.</p>
        <p>Using forward genetic approaches, we identified neurons of a third visual pathway in which the first order interneurons L3 provides a key input to direction-selective neurons via the medulla neuron Tm9. Neurons of this pathway are behaviorally required for OFF motion detection and form a novel, parallel OFF pathway. Using in vivo two photon calcium imaging, we showed that this pathway carries sustained responses to contrast changes and exhibits receptive field properties that inform elementary motion detectors about wide regions of visual space. Thus, the two OFF pathways differ in their physiological properties and the first order interneurons L2 and L3 already provide fast and slow inputs to OFF edge motion detection. We are currently investigating the mechanisms that shape the distinct properties of the OFF pathway. Our goal is to understand the cellular mechanisms, circuits and computations that implement behavioral responses to visual motion.</p>

        <p></p>
        <hr />

        <p><strong>Lunch 12:30 PM - 2:00 PM</strong></p>

        <p></p>
        <hr />

        <p><strong>Afternoon Session: Visuomotor Circuits and Internal Representations (2:00 PM - 3:30 PM)</strong></p>
        <p></p>
        <p><strong>Chair: Andreas S. Thum</strong></p>
        <p></p>


        <hr />

        <p>2:00 PM - 2:45 PM</p>
        <p><strong>Quantitative Predictions in a Drosophila Visuomotor Network</strong></p>
        <p><a href="http://www.rockefeller.edu/research/faculty/labmembers/GabyMaimon/">Anmo J. Kim</a>, Rockefeller University, New York.</p>
        <p>Vision influences behavior, but ongoing behavior also modulates vision from insects to primates. For example, we constantly move our eyes from one point to another, even as we view a static scene. These eye movements cause the whole visual image to shift on our retinas; yet we hardly notice they are happening. The same visual motion, if replayed, causes a strong sense of visual motion. Consistent with this behavioral observation, previous studies reported transient modulation of visual processing during rapid eye movements. However, the function and biophysical mechanisms of most such modulations remain unresolved. We investigated a function for behavioral modulations of visual processing in Drosophila by combining behavioral genetics, electrophysiology, and high-speed videography. We provide evidence that, via a genetic inactivation experiments, a set of motion-sensitive visual neurons regulate gaze-stabilizing head movements. We describe how, during flight turns, Drosophila perform a set of head movements that require silencing their gaze-stability reflexes along the primary rotation axis of the turn. Consistent with this behavioral requirement, we find pervasive motor-related inputs to the visual neurons, which quantitatively silence their predicted visual responses to rotations around the relevant axis while preserving sensitivity around other axes. This work proposes a function for a behavioral modulation of visual processing and illustrates how the brain can remove one sensory signal from a circuit carrying multiple related signals.</p>

        <p></p>
        <hr />

        <p>2:45 PM - 3:30 PM</p>
        <p><strong>An Internal Representation of Walking Movements in a Visual Area of the Drosophila Brain</strong></p>
        <p><a href="http://www.neuro.fchampalimaud.org/en/research/group/Chiappe/">M. Eugenia Chiappe</a>, Champalimaud Centre for the Unknown, Lisbon.</p>
        <p>Animals move their body in specific, coordinated, and flexible ways during locomotion, and this high-performance control depends on an internal estimate of self-movement. Self-movement estimation is also critical for the proper interpretation of external sensory information that may guide locomotive behaviors. This internal representation is thought to arise from the distributed activity of sensorimotor circuits; however, it is still unclear what circuits are involved, and how they implement motor-sensory coordination. Here we will discuss our initial attempts to uncover these issues in Drosophila melanogaster by performing simultaneous recordings of the fly’s walking behavior and the activity of a group of optic-flow processing neurons, the HS cells. Our results show that HS cells encode information about the fly’s walking movements in darkness, suggesting the presence of extra-retinal signals in this network. HS cells show direction selective responses driven by the turning direction of the fly, which cooperate with the cells’ canonical (visual) direction selectivity under visual stimulation. We propose that the observed convergence between visual and motor-related information during walking in HS cells creates a non-ambiguous, quantitative central representation of the movement of the body through space. This accurate representation may guide the fly’s forward movements during explorative walking. Ongoing experiments in Virtual Worlds for freely walking flies are testing this functional hypothesis.</p>

        <p></p>
        <hr />

        <p><strong>Afternoon Break 3:30 PM - 4:00 PM</strong></p>

        <p></p>
        <hr />

        <p><strong>Afternoon Session: Neural Circuit Encoding and Computation (4:00 PM - 5:30 PM)</strong></p>
        <p></p>
        <p><strong>Chair: Vijay Balasubramanian</strong></p>
        <p></p>


        <hr />

        <p>4:00 PM - 4:45 PM</p>
        <p><strong>Neural Circuits Encoding Wind Direction in Drosophila</strong></p>
        <p><a href="http://www.nagellab.com/people/">Katherine I. Nagel</a>, NYU Medical School.</p>
        <p>Wind direction is an important cue for navigation in many insect species. Drosophila are known to detect wind direction using antennal mechanoreceptors (Yorozu et al., 2009) but the central circuits downstream of these mechanoreceptors are not known. Using a novel behavioral paradigm, we show that walking flies use wind information derived from antennal mechanoreceptors to guide olfactory navigation behavior. Next, we use whole cell recordings to identify central neurons downstream of the antennae that encode wind direction information. We show that two groups of putative second-order neurons encode motion of the ipsilateral antenna, and have opposing direction tuning. We further identify a group of putative third-order neurons that exhibit strong directional tuning for wind stimuli. These neurons project both ispilaterally and contralaterally, where single-cell fills suggest they contact their sister cells in the opposite hemisphere. Antennal manipulations demonstrate that these neurons combine information from the two antennae nonlinearly to increase the dynamic range and discriminability of wind direction signals. Our findings suggest an ascending “wind pathway” that extracts wind direction information from positional signals derived from the two antennae.</p>

        <p></p>
        <hr />

        <p>4:45 PM - 5:30 PM</p>
        <p><strong>What has balance ever done for us?</strong></p>
        <p><a href="https://www.dpag.ox.ac.uk/research/vogels-group">Tim P. Vogels</a>, Department of Physiology, Anatomy and Genetics, University of Oxford.</p>
        <p></p>

        <p></p>
        <hr />


        <h4>More information about BCMC 2017 can be found <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2017">here.</a></h4>
        <br>

      </div>
    </div>


  <footer class="footer">
  <div class="row">
    <p>Copyright &copy; 2016 <a href="licenses.html#credits">FFBO Teams</a>. All Rights Reserved. <a href="licenses.html">Licenses and Credits</a></p>
  </div>
  </footer>

  <!-- jQuery Version 1.12.3 -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
  <!-- Bootstrap Core JavaScript -->
  <script src="js/bootstrap.min.js"></script>
  <!-- Custom Modernizr JavaScript -->
  <script src="js/modernizr.custom.js"></script>
  <script src="js/smooth-scroll.js"></script>
  <script src="js/jquery.stellar.min.js"></script>
  <script src="js/submenu.js"></script>
<!--  <script src="js/ffbo.js"></script>-->
</body>
</html>
