<!doctype html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8"/>
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="description" content="Fruit Fly Brain Observatory">
  <meta name="author" content="FFBO Team">

  <title>Get Involved | Workshops</title>

  <!-- Bootstrap Core CSS -->
  <link href="css/bootstrap.min.css" rel="stylesheet">
  <link href="css/ffbo.css" rel="stylesheet">
  <link rel="shortcut icon" href="ico/favicon.ico" />
</head>

<body>
  <!-- PRELOADING  -->
  <!--div id="preload">
    <div class="preload">
      <div class="loader">
      </div>
      <h2>loading ...</h2>
    </div>
  </div-->

  <!-- Navigation -->
  <nav class="navbar navbar-fixed-top navbar-ffbo top-nav-collapse" role="navigation">
      <div class="container">
          <div data-scroll-header class="navbar-header">
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#nav">
                  <span class="sr-only">Toggle navigation</span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
                  <span class="icon-bar"></span>
              </button>
              <a class="navbar-brand" href="index.html"><span><img width="50px" height="50px" src="img/ffbo_logo.png"></span> Fruit Fly Brain Observatory</a>
          </div>
          <div class="collapse navbar-collapse" id="nav">
              <ul class="nav navbar-nav navbar-right uppercase">
                  <li>
                    <a class="dropdown-toggle" role="button" data-toggle="dropdown" href="#">Get Started</a>
                    <ul class="dropdown-menu" role="menu">
                      <li class="dropdown-submenu">
                          <a href="#" class="dropdown-toggle" data-toggle="dropdown">About</a>
                          <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#mission-2">Background</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#overview-2">Overview</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neuronlp-2">NeuroNLP</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neurogfx-2">NeuroGFX</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#neuroapp-2">NeuroAPPs</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#under-the-hood-2">Under The Hood</a></li>
                          <li><a tabindex="-1" data-scroll href="http://fruitflybrain.org#team-2">Fruit Fly Brain Explorers</a></li>
                          </ul>
                      </li>
                      <li class="divider"></li>
                      <li class="dropdown-submenu">
                        <a tabindex="-1" href="#"  class="dropdown-toggle" data-toggle="dropdown">Get Involved</a>
                        <ul class="dropdown-menu" role="menu">
                          <li><a tabindex="-1" href="faq.html" target="_blank">FAQs</a></li>
                          <li><a tabindex="-1" href="hackathons.html" target="_blank">Hackathons</a></li>
                          <li><a tabindex="-1" href="https://lists.columbia.edu/mailman/listinfo/ffbo" target="_blank">Mailing List</a></li>
                          <li><a tabindex="-1" href="workshops.html" target="_blank">Workshops</a></li>
                          <li><a tabindex="-1" href="https://github.com/NeuralEnsemble/NeuroinformaticsTutorial/blob/master/Exercises/Exercise6_FruitFlyBrainObservatory.md" target="_blank">Tutorials</a></li>
                        </ul>
                      </li>
                    </ul>
                  </li>
                  <li><a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroNLP</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a href="https://neuronlp.fruitflybrain.org" target="_blank">NeuroNLP.Adult</a></li>
                          <li class="divider"></li>
                          <li><a href="https://neuronlp.larva.fruitflybrain.org" target="_blank">NeuroNLP.Larva</a></li>
                      </ul>
                  </li>
                  <li><a href="https://neurogfx.fruitflybrain.org" target="_blank">NeuroGFX</a></li>
                  <li>
                      <a class="dropdown-toggle" role="button" data-toggle="dropdown">NeuroAPPs</a>
                      <ul class="dropdown-menu" role="menu">
                          <li><a href="https://neuroapps.fruitflybrain.org/retinal_degeneration"  target="_blank">Retinal Degeneration</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/parkinsons/olfaction" target="_blank">Parkinson's Disease: Olfaction</a></li>
                          <li><a  href="https://neuroapps.fruitflybrain.org/parkinsons/vision" target="_blank">Parkinson's Disease: Vision</a></li>
                          <li><a href="https://neuroapps.fruitflybrain.org/epilepsy" target="_blank">Epilepsy</a></li>
                      </ul>
                  </li>
              </ul>
          </div>
          <!-- /.navbar-collapse -->
      </div>
      <!-- /.container -->
  </nav>


    <div class="container" style="padding-top:100px;padding-left:40px">
      <div class="col-lg-12">
        <h2>Columbia Workshop on Brain Circuits, Memory and Computation 2018</h2>
        <h4>BCMC 2018
        <br>March 15-16, 2018
        <br>Center for Neural Engineering and Computation
        <br>Columbia University, New York, NY, USA</h4><br>

        <h4>Overview</h4>
        <p>The goal of the workshop is to bring together researchers interested in developing executable models of
neural computation/processing of the brain of model organisms.
Of interest are models of computation that consist of elementary units
of processing using brain circuits and memory elements. Elementary units of computation/processing
include population encoding/decoding circuits with biophysically-grounded neuron models, non-linear dendritic processors for motion detection/direction selectivity, spike processing and pattern recognition neural circuits,
movement control and decision-making circuits, etc.
Memory units include models of spatio-temporal memory circuits, circuit models for memory access and
storage, etc.
A major aim of the workshop is to explore the integration of various sensory and control circuits in higher brain centers.
        </p>


        <p>A <a href="http://www.bionet.ee.columbia.edu/hackathons/ffbh/2018">Fruit Fly Brain Hackathon</a> is being conducted in conjunction with the workshop. Workshop participants are welcome to attend the hackathon.</p>
        <hr />

        <h4>Organizer and Program Chair</h4>
        <p><a href="http://www.ee.columbia.edu/~aurel">Aurel A. Lazar</a>, Department of Electrical Engineering, Columbia University.</p>

        <hr />

        <h4>Registration</h4>
        <p>Registration is free but all participants have to <a href="http://bcmc2018.eventbrite.com">register</a>. Thank you!</p>

        <hr />

        <h4>Lodging and Directions to Venue</h4>
        <p>Please follow this <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2018/directions">link</a> for lodging details and directions to the hotel and venue.</p>

        <hr />

        <h4>Invited Speakers (Confirmed)</h4>
    <p><a href="https://www.janelia.org/lab/ahrens-lab" target="_blank">Misha B. Ahrens</a>, Janelia Research Campus, Ashburn, VA
    <br />
     <hr class="docutils" />
    <p><strong>Randomness, Order, and Mystery in the Fly Mushroom Body</strong></p>
    <a href="https://www.janelia.org/lab/bock-lab" target="_blank">Davi Bock</a>, Janelia Research Campus, Ashburn, VA
    <br />
    </p>The mushroom body (MB) of the Drosophila olfactory system is critical for olfactory learning and memory. The MB on each side of the brain contains ~2,000 Kenyon cells (KCs) that receive olfactory inputs from projection neurons (PNs) in the MB calyx. Light microscopy (LM) data of PN-to-KC pairs pooled across many animals, as well as theoretical arguments, have suggested that the PN-to-KC synaptic network is completely random. However, the PN-to-KC network has never been mapped within a single animal. We used a whole-brain EM dataset to map the connections of PNs to ~20% of KCs in the MB on one side of the brain. We find that PN arbors are more tightly clustered in calyx than predicted from pooled LM data, and that KCs with neighboring output neurites receive more common PN input than predicted by chance. Thus the fly MB connectome, which has been considered to be random, has a clear structure. Further work will be needed to understand how this network structure impacts associative learning and recall in the fly brain.
    <hr class="docutils" />
    <p><strong>Development of Concurrent Retinotopic Maps in the Fly Motion Detection Circuit</strong></p>
    <a href="https://wp.nyu.edu/desplanlab/" target="_blank">Claude Desplan</a>, Center for Genomics and Systems Biology, NYU Abu Dhabi and Department of Biology, NYU.
    <br />
    </p>Understanding how complex brain wiring is produced during development is a daunting challenge. In the Drosophila optic lobe, information from each of 800 ommatidia is processed in distinct neuropiles, each subdivided into 800 matching retinotopic columns. In the motion vision pathway, T4 and T5 neurons respond to bright edge (T4) or dark edge (T5) motion. They collect information from the elementary motion detectors either in the Medulla (T4) or in the Lobula (T5). There are four T4 and four T5 subtypes, each tuned to one of four cardinal directions, back-to-front, front-to-back, up, or down, effectively establishing eight concurrent retinotopic maps in the Lobula Plate that detects wide-field motion. We will describe a mode of neurogenesis in which a neuroblast produces matching sets of four T4 and T5 neurons that are retinotopically coincident and have pair-wise opposite direction selectivity. Neuroblasts originating from a neuroepithelium expressing the signaling molecule Dpp produce T4 and T5 that detect vertical motion, while other neuroblasts originating from a neighboring region expressing the negative regulator of Dpp, Brinker, detect horizontal motion. We will show that retinotopy is an emergent characteristic of this neurogenic program that derives directly from neuronal birth order. This illustrates how simple developmental rules can implement complex neural organization.
        </p>Joint work with Filipe Pinto-Teixeira.
    <hr class="docutils" />
    <a href="http://www.fryelab.net/" target="_blank">Mark Frye</a>, UCLA, Los Angeles, CA
    <br />
    <a href="https://geffenlab.weebly.com/" target="_blank">Maria N. Geffen</a>, University of Pennsylvania, Philadelphia, PA
    <br />
    <a href="http://kazama.brain.riken.jp/" target="_blank">Hokto Kazama</a>,&nbsp;Riken Brain Science Institute, Saitama, Japan
    <br />
    <a href="http://cqb.pku.edu.cn/en/kxdw/zxjs/DonggenLuo/254007.shtml" target="_blank">Donggen Luo</a>, Peking University, Beijing, China
    <br />
    <a href="http://users.sussex.ac.uk/~tn41/" target="_blank">Thomas Nowotny</a>, University of Sussex, Brighton, UK
    <br />
     <hr class="docutils" />
    <p><strong>Reverse-Engineering Drosophila Limb Control Circuits</strong></p>
    <a href="https://ramdya-lab.epfl.ch/" target="_blank">Pavan P. Ramdya</a>, EPFL, Lausanne, Switzerland
    <br />
    </p>A shared goal of neuroscience and robotics is to understand how systems can be built to move effectively through the world. However, state-of-the-art algorithms for selecting and executing limbed behaviors in robots are still quite primitive compared with those used by animals. To inform robotic control approaches, we are investigating how the fly, Drosophila melanogaster, walks, grooms, and reaches. I will discuss how we are combining 2-photon imaging of the ventral nerve cord in behaving Drosophila with physics-based simulations and neural network modeling to uncover how flies achieve flexible limb control.
        <hr class="docutils" />
    <p><strong>Universality in Olfactory Coding of Odor Identity and Intensity</strong></p>
    <a href="https://scholar.harvard.edu/aravisamuel/home" target="_blank">Aravinthan D. T. Samuel</a>, Harvard University, Cambridge, MA
    <br />
    </p>Odor perception serves multiple purposes: to distinguish odorants, recognize the same odorant across concentrations, and determine concentration changes in space and time. How primary olfactory representations are structured to support and disentangle these distinct functions remains poorly understood. By interrogating the complete olfactory input circuit of the Drosophila larva, we have uncovered a set of simple cellular an ensemble-level strategies that allow for non-overlapping representations of odor identity and intensity. We find that the activity of each olfactory neuron scales with the concentration of any odorant via a fixed dose-response function, but varies in its activation threshold across ORNs. Furthermore, all activation thresholds are drawn from the same power law statistical distribution. These properties predict a universal scaling relationship between odor intensity and the overall ensemble response across olfactory space. We find that similar temporal response filters across odorants and ORNs extend these invariant relationships to fluctuating or turbulent environments. The activation threshold is thus the only free variable quantifying the tuning between any odorant and receptor. We conclude that the diverse tuning properties and dynamics of individual olfactory neurons function to produce a prescribed statistical outcome in the olfactory code. We propose that a set of invariances exhibited by the olfactory input circuit ? from single ORN to ensemble representations ? constitute efficient strategies for invariant and distinct representations of odor identity and intensity.
<hr class="docutils" />
<p><strong>Lessons from Simple Networks: Sensory Coding in the Drosophila Larval Visual System</strong></p>
    <a href="http://www.sprecherlab.com/simon-sprecher/" target="_blank">Simon Sprecher</a>, Department of Biology, University of Fribourg, Fribourg, Switzerland
    <br />
    </p>Neural circuits in the brain have the striking ability to encode sensory information with high precision, but also to adapt and modulate synaptic activity to allow memories to be formed. Cellular complexity in circuit organization is often a major obstacle when attempting to understand these processes. A comparably low number of neurons and powerful genetics allows us in the fruit fly Drosophila melanogaster to tackle fundamental topics in neuroscience with unprecedented precision. Combining connectomics, behavioural studies and in vivo life imaging we decipher visual and gustatory coding principles. Genetic manipulations with single-cell resolution allowed us to identify molecular mechanisms of multimodal and multisensory coding, supporting the existence of previously unknown gustatory coding principles. Similarly, by identifying the synaptic circuit map of the visual system and underlying function of individual neurons for visually guided behaviours we depict how spatial and temporal information is encoded, highlighting general principles of visual information processing.
            <hr class="docutils" />
    <p><strong>Olfactory Learning Alters Neuronal Gain in the Memory-Encoding Mushroom Body</strong></p>
    <a href="http://www.scripps.edu/tomchik/" target="_blank">Seth Tomchik</a>, Scripps Research Institute, Jupiter, Florida
    <br />
    </p>Learning and memory generate changes in neuronal responses to input stimuli, which are collectively referred to as a memory trace or engram. These alterations rely on dopaminergic circuits and downstream cAMP-dependent plasticity across diverse organisms. Yet it is not well understood how cAMP-dependent plasticity drives coherent changes in neuronal physiology that encode memory. In Drosophila, the mushroom body (MB) is critically involved in olfactory learning, and cAMP signaling molecules are necessary for normal memory in intrinsic MB neurons. The MB encodes a sparse representation of olfactory space, which is computationally advantageous for learning. Our recent studies have suggested that olfactory classical conditioning alters the gain of MB neuronal responses to sensory stimuli in a cAMP-dependent manner. This was observed particularly during appetitive conditioning, suggesting that it may either include a motivational salience component or reflect a bias toward reward learning. These data link the physiology of MB neurons to the behavioral roles for cAMP signaling molecules in learning and memory. I will discuss these results and their implications for development of models of olfactory learning and MB function.
    <hr class="docutils" />
    <a href="http://jovo.me/" target="_blank">Joshua T. Vogelstein</a>, Johns Hopkins University, Baltimore, MD
    <br />
    <a href="https://www.dpag.ox.ac.uk/research/waddell-group" target="_blank">Scott Waddell</a>, University of Oxford, Oxford, UK
    <br />
    <hr class="docutils" />
    <p><strong>Sequencing the Connectome</strong></p>
    <a href="http://zadorlab.cshl.edu/" target="_blank">Anthony M. Zador</a>, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
    <br />
</p>To study the connectivity of neural circuits at high resolution, we have developed an approach based on DNA sequencing that allows us to determine the connectivity of single neurons. The key idea is to tag each neuron with a random nucleotide sequence (a "barcode"), which can be read out by high-throughput sequencing. By recasting neuroanatomy, which is traditionally viewed as a problem of microscopy, as a problem of sequencing, MAPseq harnesses advances in sequencing to permit high-throughput interrogation of brain circuits. The ability to read out brain circuitry in a high-throughput way has the potential to accelerate our understanding of both normal brain function, and the causes underlying neuropsychiatric disorders including autism and schizophrenia.
        <hr />


        <h4>More information about BCMC 2018 can be found <a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2018">here.</a></h4>
        <br>

        <hr />

        <h4>Past BCMC workshops</h4>

        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2017">BCMC 2017</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2016">BCMC 2016</a></p>
        <p><a href="http://www.bionet.ee.columbia.edu/workshops/bcmc/2015">BCMC 2015</a></p>
        <br>

      </div>
    </div>


  <footer class="footer">
  <div class="row">
    <p>Copyright &copy; 2016 <a href="licenses.html#credits">FFBO Teams</a>. All Rights Reserved. <a href="licenses.html">Licenses and Credits</a></p>
  </div>
  </footer>

  <!-- jQuery Version 1.12.3 -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
  <!-- Bootstrap Core JavaScript -->
  <script src="js/bootstrap.min.js"></script>
  <!-- Custom Modernizr JavaScript -->
  <script src="js/modernizr.custom.js"></script>
  <script src="js/smooth-scroll.js"></script>
  <script src="js/jquery.stellar.min.js"></script>
  <script src="js/submenu.js"></script>
<!--  <script src="js/ffbo.js"></script>-->
</body>
</html>
